<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD004025</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD004025</h1>
    
	<div class="dataset-content">
	PXD004025 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Assessment of Label-Free Quantification in Discovery Proteomics and Impact of Technological Factors and Natural Variability of Protein Abundance</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Proteome wide protein quantification has become facile in the last years supported by label-free discovery proteomics approaches and powerful, easy to use software. We set out to independently evaluate this highly peptide centric technology focusing on the performance of some of the currently most popular quantitative proteomics software, Proteome Discoverer, Scaffold, MaxQuant and Progenesis QIP. The sample to sample variability introduced into protein abundance estimates by the technology itself, thereby affecting the validity of changes in protein abundance reported to be biological in nature, was determined. Using the same experimental design we were interested in the potential of shotgun proteomics to uncover biological contributions to changes in protein abundance in essentially isogenic plants grown under the same environmental conditions. The accuracy, precision, limit of quantification, intra-analysis repeatability and other performance metrics of protein quantification were determined with the softwares’ ready-to-run, default parameters and some modified settings. MaxQuant and Progenesis QIP each had decided advantages and exhibited direct proportionality between measured and actual protein abundance. The inherent variability in protein quantification in measurements of Arabidopsis thaliana proteomes was substantially higher than the biological sample to sample variability. Strikingly, correlation between Arabidopsis proteins quantified with MaxQuant and Progenesis QIP was poor due to differential protein inference. Both of these points warrant caution in the interpretation of shotgun proteomics studies. The abundance of around 99% of proteins was constant in repeated sampling of the Arabdiopsis thaliana Col-0 inbred accession; however a handful showed substantial quantitative variability. The function stress/stimulus response was highly significantly overrepresented for these proteins. Heritable information beyond genetic polymorphism that affects gene expression levels has recently been documented for these functions. We speculate on the potential of discovery proteomics to measure gene expression levels as a conduit of the epigenotype shaping quantitative traits.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2017-10-24</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD004025-3&outputMode=XML&test=no' target="_blank">Submission_2017-10-24_04:04:27.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="http://dx.doi.org/10.6019/PXD004025" target="_blank">http://dx.doi.org/10.6019/PXD004025</a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Supported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Wolfgang Hoehenwarter</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Arabidopsis thaliana (Mouse-ear cress);  NCBI TaxID: 3702; </td></tr><tr><td><b>ModificationList</b></td><td>Oxidation; Carbamidomethyl</td></tr><tr><td><b>Instrument</b></td><td>LTQ Orbitrap Velos</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2016-04-20 23:46:17</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD004025-1&test=no">1</a></td><td >2017-03-07 01:59:43</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD004025-2&test=no">2</a></td><td >2017-03-09 00:30:27</td><td >announced</td><td >Updated project metadata.</td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD004025-3&test=no"><span class='current'>&#9205;</span> 3</a></td><td class="dataset-currentrev">2017-10-24 04:04:29</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Updated project metadata.</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Al Shweiki MR, Mönchgesang S, Majovsky P, Thieme D, Trutschel D, Hoehenwarter W, Assessment of Label-Free Quantification in Discovery Proteomics and Impact of Technological Factors and Natural Variability of Protein Abundance. J Proteome Res, 16(4):1410-1424(2017) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=28217993" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>curator keyword: Technical, Biological</td></tr><tr><td>submitter keyword: Shotgun Proteomics, Label-free, Protein Inference, MaxQuant, Progenesis QIP, Scaffold, Arabidopsis, Epigenetics</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Wolfgang Hoehenwarter</th></tr>
<tr><td>contact affiliation</td><td>Leibniz Institute of Plant Biochemistry</td></tr><tr><td>contact email</td><td>wolfgang.hoehenwarter@ipb-halle.de</td></tr><tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Wolfgang Hoehenwarter</th></tr>
<tr><td>contact affiliation</td><td>Leibniz Institute of Plant Biochemistry</td></tr><tr><td>contact email</td><td>wolfgang.hoehenwarter@ipb-halle.de</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2017/03/PXD004025 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD004025 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD004025" target="_blank">PXD004025</a><ol><li>Label: PRIDE project</li><li>Name: Assessment of Label-Free Quantification in Discovery Proteomics and Impact of Technological Factors and Natural Variability of Protein Abundance</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>