<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD009840</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD009840</h1>
    
	<div class="dataset-content">
	PXD009840 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Mining Missing Protein Landscape in Subcellular Proteome of Human Embryonic Stem Cells</td></tr><tr><td><b>Description</b></td><td class='breakwords'> Human embryonic stem cells (hESCs) have capacity of self-renewal and multi-lineage differentiation into derivatives of three germ layers that are of clinical importance for regeneration medicine. Despite the significant progress of hESCs study for clinical application, factors responsible for maintaining the undifferentiated and pluripotent nature of hESCs are still largely unknown. The knowledge barrier between human genomic information and its proteomics was proposed due to the ‘missing proteins’(MPs), proteins that do not have a stringent evidence for existence at protein level, in away investigation of MPs has been continued since 2012 as one of the major goals of Chromosome-Centric Human Proteome Project (C-HPP). Nearly 18% of the human proteins are awaiting high-stringency mass spectrometry evidence of their existences in the human proteome. In this study, we proposed subcellular proteomics analysis aimed to identify the nuclear, cytoplasmic and membrane proteins involved in regulating ESC pluripotency and among them to mine missing proteins in the very early cell status. Our study achieved an identification of 11,970 unique proteins (264,088 unique peptides) in hESCs of which 6,138 were annotated as membrane proteins. Compared with large scale proteomic analysis of hESCs this study additionally identified 3,626 unique protein coding genes. This large scale proteomic data mined a total of 296 candidate detections of missing proteins at 1% FDR. Following the strict criteria of HPP data interpretation guidelines (version 2.1.0), a total of 25 Gold Missing Proteins (GMPs) were confidently identified among which 12 of them were verified with synthetic peptides and finally 10 were confirmed with 2 unique peptides. In summary, these analyses suggested that our results covered the most comprehensive proteome and membrane proteomics of hESCs that would be valuable to provide protein-level evidence for missing membrane proteins. 
Sample Processing Protocol: 
A subcellular purification was applied to generate nuclear, cytoplasmic and membrane protein fractions using our previously reported 2-step centrifugation method with sucrose. These three protein fractions were subjected to gel-assisted digestion with trypsin followed by peptide fractionation. We employed our developed sensitive high-pH reversed phase (Hp-RP) stop-and-go extraction tip (StageTip) technique to collect 7 RP fractions for the first batch analysis. For the second batch of membrane proteome analysis and both batches of cytoplasmic and nuclear proteomes, we applied High pH reversed-phase liquid chromatography (RPLC) and concatenation to generate 12 (for membrane proteome) and 24 (for nucleus and cytoplasm proteomes) RP fractions, respectively. All the RP fractions were analyzed in duplicate by high-end Orbitrap Fusion™ or Fusion™ Lumos™ Tribrid™ mass spectrometers.
Data Processing protocol: 
Protein identifications were performed for each RP fraction by using Proteome Discoverer 2.1 (PD 2.1) with three search engines, Mascot, SequestHT and MSAmanda, at 1% FDR in peptide, protein and and high confidence PSM levels. To obtain the total list of identified proteins in nuclear, cytoplasmic and membrane proteomes, we adopted the consensus workflow in PD 2.1 to combine all the fraction data with filtering to 1% protein FDR. In order to mine missing proteins from the subcellular proteomes of hECs, we followed CHPP guideline version 2.1.0 with stringent criteria among which (1) detection at 1% FDR (protein, peptide) and with high confidence, unambiguous PSMs; (2) uniqueness of the peptide, alternate peptide mappings, isobaric sequences or SAAVs, checked by the Peptide Uniqueness Checker (nextprot, 20180117 release); (3) unique peptide sequences of length ≥9 amino acid; (4) two or more unique, non-nested peptide sequences were applied entirely for stepwise filtering of candidate detections. For validating missing proteins with synthetic peptides, MRM analysis was conducted by QTrap 5500 system (AB SCIEX Concord, ON) using Trap column (ACQUITY UPLC® symmetry C18, 5 µm, 180 µm x 20mm) with BEH C18-AQ column (nanoACQUITY® 1.7 µm, 75 µm x 250 mm). Membrane protein annotations were used from neXtProt 2018-01 consisting of Human Proteome 
We further annotated the cellular localizations and the number of TMH for all identified proteins using the latest version of neXtProt (2018-01). Total identified proteins in three subcellular proteome data were analyzed using KEGG & IPA databases for contracting pathway analysis. The Human Proteome Atlas (HPA), Gene Ontology and other informatic tools were used as a source for generating lineage specific analysis.
</td></tr><tr><td><b>HostingRepository</b></td><td>jPOST</td></tr><tr><td><b>AnnounceDate</b></td><td>2019-05-20</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD009840.0-3&outputMode=XML&test=no' target="_blank">Submission_2021-02-04_17:31:03.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Mehari Weldemariam</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Homo sapiens (Human);  NCBI TaxID: 9606; </td></tr><tr><td><b>ModificationList</b></td><td>L-cysteine methyl disulfide; L-methionine sulfoxide; deamidated L-asparagine; deamidated L-glutamine</td></tr><tr><td><b>Instrument</b></td><td>LTQ Orbitrap</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2018-05-19 08:43:43</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD009840-1&test=no">1</a></td><td >2018-10-09 18:03:07</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD009840-2&test=no">2</a></td><td >2018-10-09 22:12:34</td><td >announced</td><td >Updated PubMed.</td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD009840-3&test=no"><span class='current'>&#9205;</span> 3</a></td><td class="dataset-currentrev">2021-02-04 17:31:04</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">2021-02-04: Updated FTP location.</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Weldemariam MM, Han CL, Shekari F, Kitata RB, Chuang CY, Hsu WT, Kuo HC, Choong WK, Sung TY, He FC, Chung MCM, Salekdeh GH, Chen YJ, Subcellular Proteome Landscape of Human Embryonic Stem Cells Revealed Missing Membrane Proteins. J Proteome Res, 17(12):4138-4151(2018) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=30203655" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: Subcellular fractionation, membrane proteins, human embryonic stem cells, missing protein</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Yu-Ju Chen</th></tr>
<tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Mehari Weldemariam</th></tr>
<tr><td>contact affiliation</td><td>Institute of Chemistry, Academia Sinica</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=https://repository.jpostdb.org/entry/JPST000421 target="_blank">jPOST dataset URI</a></td></tr><tr><td><a href=ftp://ftp.biosciencedbc.jp/archive/jpostrepos/JPST000421 target="_blank">Dataset FTP location</a></td></tr></table>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>