<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD010262</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD010262</h1>
    
	<div class="dataset-content">
	PXD010262 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>MudPIT analyses of the chromatin associated proteins isolated from the nucleus of Plasmodium falciparum at three time-points of the intraerythrocytic development cycle</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Parasite cultures- Plasmodium falciparum strain 3D7 was cultured in human O+ erythrocytes at 5% hematocrit. Cultures were synchronized at ring stage with 5% (w/v) D-sorbitol treatments. Parasite cultures (8% parasitemia in 5% hematocrit) were harvested 48 hours after the first sorbitol treatment (ring stage) and 18 hours (trophozoite stage), and 36 hours thereafter (schizont stage).
Chromatin enrichment for proteomics (ChEP)- Chromatin-associated proteins (CAPs) were isolated at three different stages of the parasite erythrocytic cycle: early ring (0 hours post infection), early trophozoite (18-hpi), and late schizont stages (36-hpi). Synchronized parasites were cross-linked with 1% formaldehyde for 15 min at 37oC. Cross-linking was quenched by adding 0.125 M glycine for 5 min at room temperature. The parasites were then washed with phosphate-buffered saline (PBS), incubated in nuclear extraction buffer (10 mM KCl, 0.1 mM EDTA, 0.1 mM EGTA, 1 mM DTT, 0.5 mM AEBSF, protease inhibitor cocktail (Roche) and 0.25% Igepal) for 30 min and needle sheared using a 25-gauge needle. Extracted nuclei were spun at 1,300 rcf for 20 min at 4oC. The nuclear pellet was washed with highly denaturing extraction buffers containing 4% SDS and 8M urea to wash away non-cross-linked proteins. Chromatin was solubilized and genomic DNA was sheared by sonication (Covaris; 5% duty cycle, 140 intensity peak incident power, 200 cycles per burst).
Cytoplasmic fractions- As a negative control, protein from the cytoplasmic fractions isolated from parasites at the same time-points were also extracted. Synchronized parasite cultures were collected and subsequently lysed by incubating in 0.15% saponin for 10 min on ice.  Parasites were centrifuged at 1,500 rcf for 10 min at 4oC and washed three times with PBS.  For each wash, parasites were resuspended in cold PBS and centrifuged for 10 min at 1,500 rcf at 4oC.  After the last wash, parasites were resuspended in PBS, transferred to a microcentrifuge tube, and centrifuged for 5 min at 2100 rcf at 4oC.  Subsequently, the parasite pellet was resuspended in 1.5X volume of cytoplasmic lysis buffer (0.65% Igepal CA-360 (Sigma-Aldrich), 10 mM Tris-HCl pH 7.5, 150 mM NaCl, 1 mM EDTA, 1 mM EGTA, 2 mM 4-(2-aminoethyl)benzenesulfonyl fluoride hydrochloride (AEBSF), and EDTA-free protease inhibitor cocktail (Roche) and lysed by passing through a 26G 1/2-inch needle fifteen times.  Parasite nuclei were centrifuged at 14,500 rcf for 15 min at 4oC and the supernatant containing the cytoplasmic extract was collected.
Multidimensional Protein Identification Technology- Two biological replicates with two technical replicates each were prepared for ChEP and cytoplasmic control samples at the ring, trophozoite, and schizont stages. Proteins were precipitated with 20% trichloroacetic acid (TCA) and the resulting pellet was washed once with 10% TCA and twice with cold acetone. About 50 ug of the TCA-precipitated protein pellet was solubilized using Tris-HCl pH 8.5 and 8 M urea, followed by addition of TCEP (Tris(2-carboxyethyl)phosphine hydrochloride; Pierce) and CAM (chloroacetamide; Sigma) were added to a final concentration of 5 mM and 10 mM, respectively. Proteins were digested using Endoproteinase Lys-C at 1:100 w/w (Roche) at 37oC overnight. The samples were brought to a final concentration of 2 M urea and 2 mM CaCl2 and a second digestion was performed overnight at 37oC using trypsin (Promega) at 1:100 w/w. The reactions were stopped using formic acid (5% final). The samples were loaded on a split-triple-phase fused-silica micro-capillary column and placed in-line with a linear ion trap mass spectrometer (LTQ, Thermo Scientific), coupled with a Quaternary Agilent 1100 Series HPLC system. A fully automated 10-step chromatography run was carried out. Each full MS scan (400-1600 m/z) was followed by five data-dependent MS/MS scans. The number of the micro scans was set to 1 both for MS and MS/MS. The settings were as follows: repeat count 2; repeat duration 30 s; exclusion list size 500 and exclusion duration 120 s, while the minimum signal threshold was set to 100. The MS/MS data set was searched using ProLuCID (v. 1.3.3) against a database consisting of 5,530 P. falciparum non-redundant proteins (PlasmoDB-34), 36,628 Homo sapiens NR proteins (NCBI -released June 10, 2016), 193 usual contaminants, and, to estimate false discovery rates (FDRs), 42,351 randomized amino acid sequences derived from each NR protein entry. To account for alkylation by CAM, 57 Da were added statically to the cysteine residues. To account for the oxidation of methionine to methionine sulfoxide, 16 Da were added as a differential modification to the methionine residue. Peptide/spectrum matches were sorted and selected to an FDR less than 5% at the peptide and protein levels, using DTASelect in combination with swallow, an in-house software.</td></tr><tr><td><b>HostingRepository</b></td><td>MassIVE</td></tr><tr><td><b>AnnounceDate</b></td><td>2019-09-16</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD010262.0-1&outputMode=XML&test=no' target="_blank">Submission_2019-09-16_10:45:52.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Non peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Laurence Florens</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Plasmodium falciparum 3D7;  NCBI TaxID: 36329; </td></tr><tr><td><b>ModificationList</b></td><td>S-carboxamidomethyl-L-cysteine; L-methionine sulfoxide</td></tr><tr><td><b>Instrument</b></td><td>LTQ</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2018-06-28 14:35:55</td><td >ID requested</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD010262-1&test=no"><span class='current'>&#9205;</span> 1</a></td><td class="dataset-currentrev">2019-09-16 10:45:53</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev"></td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>no publication</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: Chromatin Enrichment for Proteomics (ChEP), Chromatin-Associated Proteins (CAPs), Chromatin Structure , Genome Architecture </td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Laurence Florens</th></tr>
<tr><td>contact affiliation</td><td>The Stowers Institute for Medical Research</td></tr><tr><td>contact email</td><td>laf@stowers.org</td></tr><tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Laurence Florens</th></tr>
<tr><td>contact affiliation</td><td>Stowers Institute for Medical Research</td></tr><tr><td>contact email</td><td>laf@stowers.org</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=http://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=168daea806574df1b8b7a9373a4c92d8 target="_blank">MassIVE dataset URI</a></td></tr><tr><td><a href=ftp://massive.ucsd.edu/MSV000082520/ target="_blank">Dataset FTP location</a></td></tr></table>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>