<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD000054</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD000054</h1>
    
	<div class="dataset-content">
	PXD000054 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Proteogenomic analysis of Helicobacter pylori strain 26695</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Dear Sir or Madam,  we report an in-depth proteogenomics study of Helicobacter pylori strain 26695 and provide the supporting MS data via ProteomExchange. The study includes 2 biological replicates with 6 different datasets:  G1: in-gel digestion with trypsin, replicate 1 G2: in-gel digestion with trypsin, replicate 2 T1: SEC fractionation of low molecular weight (LMW) proteins and subsequent trypsin digestion, replicate 1 T2: SEC fractionation of LMW proteins and subsequent trypsin digestion, replicate 2 A1: SEC fractionation of LMW proteins and subsequent AspN digestion, replicate 1 A2: SEC fractionation of LMW proteins and subsequent AspN digestion, replicate 2 L1: SEC fractionation of LMW proteins and subsequent LysC digestion, replicate 1 L2: SEC fractionation of LMW proteins and subsequent LysC digestion, replicate 2  In our proteogenomics approach, we could identify four previously missing protein annotations and were able to correct sequences of six protein coding regions. Furthermore we identified signal peptidase cleavage sites for 72 different proteins. MGFs were generated by Maxquant 1.1 [1] using recalibration of peptide parent masses. For PRIDE (http://www.ebi.ac.uk/pride) submission, we made an additional database search with Mascot and X!Tandem using the SearchGUI [2]. Therefore we searched against a NCBI database of H. pylori strain 26695 complemented with the sequence corrections, signal peptide cleavage sites and missing annotations with the same configurations as described in materials and methods. For pride xml export we used the software PeptideShaker (http://code.google.com/p/peptide-shaker/). The complemented database has entries which will be submitted to the UniProtKB via SPIN. The entries have the according SPIN number as accession number. The NCBI accession numbers for the shortened sequences due to signal peptide cleavage are extended with “_1”. The fasta database is added to the submission. For additional information, please contact me: stephan.mueller@ufz.de  Yours sincerely, Stephan Mueller  References: [1] Cox J, Neuhauser N, Michalski A, Scheltema RA, Olsen JV, Mann M. Andromeda: a peptide search engine integrated into the MaxQuant environment. Journal of proteome research. 2011;10:1794-805. [2] Vaudel M, Barsnes H, Berven FS, Sickmann A, Martens L. SearchGUI: An open-source graphical user interface for simultaneous OMSSA and X!Tandem searches. Proteomics. 2011;11:996-9.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2017-10-24</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD000054-3&outputMode=XML&test=no' target="_blank">Submission_2017-10-24_00:12:57.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="http://dx.doi.org/10.6019/PXD000054" target="_blank">http://dx.doi.org/10.6019/PXD000054</a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Supported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Stephan MÃ¼ller</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Helicobacter pylori 26695;  NCBI TaxID: 85962; </td></tr><tr><td><b>ModificationList</b></td><td>acetylated residue: 42.010565; 2-pyrrolidone-5-carboxylic acid (Glu): -18.010565; 2-pyrrolidone-5-carboxylic acid (Gln): -17.026549; iodoacetamide derivatized amino-terminal residue: 57.021464; monohydroxylated lysine: 15.994915</td></tr><tr><td><b>Instrument</b></td><td>instrument model: LTQ-Orbitrap; LTQ Orbitrap</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2012-10-18 03:43:06</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000054-1&test=no">1</a></td><td >2013-08-01 04:51:42</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000054-1&test=no">1</a></td><td >2014-07-28 00:24:13</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000054-2&test=no">2</a></td><td >2013-08-01 09:21:12</td><td >announced</td><td >Add reference</td></tr>
<tr><td ><a href="GetDataset?ID=PXD000054-2&test=no">2</a></td><td >2017-06-21 00:30:31</td><td >announced</td><td >Updated project metadata.</td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD000054-3&test=no"><span class='current'>&#9205;</span> 3</a></td><td class="dataset-currentrev">2017-10-24 00:12:58</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Updated project metadata.</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Müller SA, Findeiß S, Pernitzsch SR, Wissenbach DK, Stadler PF, Hofacker IL, von Bergen M, Kalkhof S, Identification of new protein coding sequences and signal peptidase cleavage sites of Helicobacter pylori strain 26695 by proteogenomics. J Proteomics, 86():27-42(2013) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=23665149" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: Proteogenomics, Proteomics, Helicobacter pylori strain, Signal peptide, Coding sequence, LTQ Orbitrap</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Stephan MÃ¼ller</th></tr>
<tr><td>contact affiliation</td><td>Proteomics</td></tr><tr><td>contact email</td><td>stephan.mueller@ufz.de</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2013/08/PXD000054 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD000054 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD000054" target="_blank">PXD000054</a><ol><li>Label: PRIDE project</li><li>Name: Proteogenomic analysis of Helicobacter pylori strain 26695</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>