<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD010446</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD010446</h1>
    
	<div class="dataset-content">
	PXD010446 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Sequential Fractionation Strategy Identifies Nine Missing Proteins in The Mitochondrial Proteome of Commonly Used Cell Lines</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Identification of mitochondrial proteins with a bottom up approach after organelle enrichment and 1D PAGE separation for 5 different cell lines. The bands were cut and the proteins reduced, alkylated and digested with trypsin according to a conventional protocol. Acquisition were performed on the Orbitrap Fusion Tribrid mass spectrometer in top speed mode with 3 seconds cycles.
Raw data were processed using PEAKS Studio 7.5 (Bioinformatics Solutions Inc.) and searched using the PEAKS search engine and the SPIDER peptide mutation and homology search tool against neXtProt (July 2017; 42,151 total entries). Parent Mass Error Tolerance was set to 10.0 ppm and Fragment Mass Error Tolerance to 0.6 Da. Other search parameters were trypsin enzyme specificity, two missed cleavages per peptide, fixed Carbamidomethylation of Cys and variable Oxidation of Met, Deamidation of Gln and Asn (NQ), Phosphorylation of Ser, Thr and Tyr and Acetylation of Lys with two variable PTM per peptide. Non-specific cleavage was allowed to only one end of the peptide. FDR estimation was enabled, and precursor options corrected. To follow the HPP Mass Spectrometry Data Interpretation Guidelines Version 2.1 and in view of our goal of finding missing proteins, we set the FDR threshold on PSMs to 2%, typically resulting in FDR on peptides lower than 5% and we filtered out all the proteins identified with only one peptide, resulting in a FDR at the protein level of 0.0%.</td></tr><tr><td><b>HostingRepository</b></td><td>MassIVE</td></tr><tr><td><b>AnnounceDate</b></td><td>2018-10-12</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD010446-1&outputMode=XML&test=no' target="_blank">Submission_2018-10-12_07:49:23.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Supported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Maurizio Ronci</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Homo sapiens;  common name: human;  NCBI TaxID: 9606; </td></tr><tr><td><b>ModificationList</b></td><td>Carbamidomethyl; Oxidation; Deamidated; Phospho; Acetyl</td></tr><tr><td><b>Instrument</b></td><td>Orbitrap Fusion</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2018-07-16 09:04:23</td><td >ID requested</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD010446-1&test=no"><span class='current'>&#9205;</span> 1</a></td><td class="dataset-currentrev">2018-10-12 07:49:24</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev"></td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Ronci M, Pieroni L, Greco V, Scotti L, Marini F, Carregari VC, Cunsolo V, Foti S, Aceto A, Urbani A, Sequential Fractionation Strategy Identifies Three Missing Proteins in the Mitochondrial Proteome of Commonly Used Cell Lines. J Proteome Res, 17(12):4307-4314(2018) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=30284448" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: mt-HPP,  missing proteins,  mitochondrial proteome,  protein fractionation,  high-density datasets</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Maurizio Ronci</th></tr>
<tr><td>contact affiliation</td><td>University G. d'Annunzio</td></tr><tr><td>contact email</td><td>maurizio.ronci@unich.it</td></tr><tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Maurizio Ronci</th></tr>
<tr><td>contact affiliation</td><td>University G. d'Annunzio</td></tr><tr><td>contact email</td><td>maurizio.ronci@unich.it</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=http://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=d3b4ae451f8c49468675513ebff2624d target="_blank">MassIVE dataset URI</a></td></tr><tr><td><a href=ftp://massive.ucsd.edu/MSV000082409 target="_blank">Dataset FTP location</a></td></tr></table>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>