<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD004349</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD004349</h1>
    
	<div class="dataset-content">
	PXD004349 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Histone modifications in Plasmodium falciparum at 30 hours post-invasion</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Synchronized Plasmodium falciparum-infected human blood cultures were harvested at seven time points along the asexual intra-erythrocytic parasite life cycle: 0 hours post-invasion, 6 hpi, 12 hpi, 18 hpi, 24 hpi, 30 hpi and 36 hpi.  Plasmodium histones were extracted using the standard hydrochloric acid method and precipitated using trichloroacetic acid.  Two biological replicates were digested with three enzymes of diverse specificities: endoproteinase LysC + trypsin (Ti), endoproteinase LysC + GluC  (KCEC), and endoproteinase ArgC (RC). The resulting peptide mixtures were analyzed by Multidimensional Protein Identification Technology (MudPIT) coupled to high-resolution tandem mass spectrometry on an LTQ Orbitrap.

RAW files were extracted into .ms2 file format using RawDistiller v. 1.0 (in-house developed software). After offline recalibration, the MS/MS spectra were first searched using SEQUEST v.27 (rev.9) with a peptide mass tolerance of 10 ppm and of +/- 0.5 amu for fragment ions. No enzyme specificity was imposed during the SEQUEST search against a protein database combining 5,439 non-redundant Plasmodium falciparum (PlasmoDB release 5.5) and 30,552 human proteins (NCBI 2008-03-04 release, as well as 162 usual contaminants such as human keratins, IgGs and proteolytic enzymes. To estimate false discovery rates (FDR), each protein sequence was randomized (keeping the same amino acid composition and length) and the resulting &quot;shuffled&quot; sequences were added to the database used for the SEQUEST searches, for a total search space of 72,306 amino acid sequences. To account for alkylation by CAM, 57.02146 Da were added statically to cysteine residues for all searches.

We set up 44 searches to query for peptides containing various combinations of methylated lysines and arginines (+14.0157), oxidized methionines and hydroxylated lysines, prolines, aspartates, histidines, and tyrosines (+15.9949), formylated lysines and arginines (+27.9949), dimethylated lysines and arginines (+28.0314), trimethylated lysines (+42.0470), acetylated lysines, serines and threonines (+42.0106), crotonylated lysines (+68.0261), phosphorylated serines, threonines, and tyrosines (+79.9663), ubiquitinated lysines (+114.0429), as well as potentially N-terminally acetylated methionines, serines, alanines, and valines (+42.0106), and oxidized methionines (+58.0055). The maximum number of modified amino acids per differential modification in a peptide was limited to six. 

The MS/MS dataset was also searched using the same parameters with OMSSA to validate newly described PTMs.
</td></tr><tr><td><b>HostingRepository</b></td><td>MassIVE</td></tr><tr><td><b>AnnounceDate</b></td><td>2016-06-16</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD004349-1&outputMode=XML&test=no' target="_blank">Submission_2016-06-16_06:03:23.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Laurence Florens</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Plasmodium falciparum 3D7;  NCBI TaxID: 36329;  scientific name: Homo sapiens;  common name: human;  NCBI TaxID: 9606; </td></tr><tr><td><b>ModificationList</b></td><td>N-acetyl-L-methionine; L-methionine sulfoxide; unknown modification: [M] [58.0055] N-acetyl-L-methionine sulfoxide; unknown modification: [K] [15.994915] 5-hydroxy-L-lysine; unknown modification: [K] [42.010565] N6-acetyl-L-lysine; unknown modification: [K] [14.015650] N6-methyl-L-lysine; unknown modification: [K] [28.031300] N6,N6-dimethyl-L-lysine; Trimethyl; N6-formyl-L-lysine; N6-crotonyl-L-lysine; ubiquitination signature dipeptidyl lysine; N5-methyl-L-arginine; symmetric dimethyl-L-arginine; formylated residue; O-phospho-L-serine; N-acetyl-L-serine; O-phospho-L-threonine; N-acetyl-L-threonine; O4'-phospho-L-tyrosine; 3'-hydroxylation of L-tyrosine to L-3',4'-dihydroxyphenylalanine; monohydroxylated proline; 3-hydroxy-L-histidine; 3-hydroxy-L-aspartic acid; N-acetyl-L-alanine; N-acetyl-L-valine; S-carboxamidomethyl-L-cysteine</td></tr><tr><td><b>Instrument</b></td><td>LTQ Orbitrap</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2016-06-16 06:03:21</td><td >ID requested</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD004349-1&test=no"><span class='current'>&#9205;</span> 1</a></td><td class="dataset-currentrev">2016-06-16 06:03:24</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev"></td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Saraf A, Cervantes S, Bunnik EM, Ponts N, Sardiu ME, Chung DW, Prudhomme J, Varberg JM, Wen Z, Washburn MP, Florens L, Le Roch KG, Dynamic and Combinatorial Landscape of Histone Modifications during the Intraerythrocytic Developmental Cycle of the Malaria Parasite. J Proteome Res, 15(8):2787-801(2016) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=27291344" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: Histones, Post-translational Modifications, malaria parasite, intra-erythrocytic cycle, MudPIT</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Laurence Florens</th></tr>
<tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Laurence Florens</th></tr>
<tr><td>contact affiliation</td><td>Stowers Institute for Medical Research</td></tr><tr><td>contact email</td><td>laf@stowers.org</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=http://massive.ucsd.edu/ProteoSAFe/status.jsp?task=18d8ca13ac694d189aab12a70f0e6b1c target="_blank">MassIVE dataset URI</a></td></tr><tr><td><a href=ftp://massive.ucsd.edu/MSV000079230 target="_blank">Dataset FTP location</a></td></tr></table>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>