<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD000284</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD000284</h1>
    
	<div class="dataset-content">
	PXD000284 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Comparative proteomic analysis of human bile from malignant and nonmalignant biliary stenosis</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Four bile samples collected from patients with a malignant (PAC, CC) or a nonmalignant (CP, BS) biliary stenosis were used for comparative proteomic analysis. Briefly, bile samples were centrifuged at 16,000/g/ for 10 min at 4 C. Each supernatant was delipided with Cleanascite (Biotech Support Group, North Brunswick, NJ, USA) and ultrafiltrated using a 3 kDa filter cut-off (YM-3 centricon, Millipore, Bedford, MA, USA). For each sample, 50 ug of proteins were mixed with 0.5 M triethylammonium bicarbonate (TEAB) buffer pH 8.0 to a total volume of 100 uL. One ug of bovine lactoglobulin (LACB) was spiked in each sample as an internal standard. Proteins were reduced and alkylated before digestion of N-linked oligosaccharides from glycoproteins using PNGase F (Sigma-Aldrich, St. Louis, MO, USA). Proteins were then digested with trypsin and the resulting peptides were tagged with one of the isobaric tags from the iTRAQ reagents 4plex kit (AB Sciex, Foster City, CA, USA) according to the manufacturer's instructions. The four samples were then pooled and dried under vacuum. Labeled peptides were fractionated according to their isoelectric point on an Agilent 3100 OFFGEL fractionator using 24 cm pH 3-10 linear immobilized pH gradients (GE Healthcare, Chalfont St. Giles, UK) following manufacturer's instructions. Fractions were then recovered in separate tubes for high-performance liquid chromatography-tandem mass spectrometry (HPLC-MS/MS). MS analysis was performed on a LTQ Orbitrap XL from Thermo Electron (San Jose, CA, USA) equipped with a NanoAcquity HPLC system from Waters. Peptides were trapped on a home-made 5 um 200 A Magic C18 AQ (Michrom, Auburn, CA, USA) 0.1 x 20 mm pre-column and separated on a home-made 5 um 100 A Magic C18 AQ (Michrom) 0.75 x 150 mm column with a gravity-pulled emitter. The analytical separation was run for 65 min using a gradient of H_2 O/formic acid (FA) 99.9%/0.1% (solvent A) and CH_3 CN/FA 99.9%/0.1% (solvent B). The gradient was run at a flow rate of 220 nL/min as follows: 5% B for 1 min, from 5 to 35% B in 54 min, from 35 to 80% B in 10 min. For MS survey scans, the OT resolution was set to 60,000 and the ion population was set to 5 x 10^5 with an m/z window from 400 to 2000. Maximum of 3 precursors were selected for both collision-induced dissociation (CID) in the LTQ and high-energy C-trap dissociation (HCD) with analysis in the Orbitrap (OT). For MS/MS in the LTQ, the ion population was set to 1 x 10^4 (isolation width of 2 m/z) while for MS/MS detection in the OT, it was set to 2 x 10^5 , with resolution of 7,500, first mass at m/z = 100, and maximum injection time of 750 ms. The normalized collision energies were set to 35% for CID and 60% for HCD. Peak lists from MS analysis were searched against Uniprot_Swissprot database (version 2011_2) using Phenyx protein identification software (GeneBio, Geneva, Switzerland). /Homo Sapiens/ taxonomy was specified for database searching. The parent ion tolerance was set to 10 ppm. Variable amino acid modifications were oxidized methionine and deamidated asparagine. Carbamidomethylation of cysteines and iTRAQ-labeled peptides on the amino terminus and lysine were set as fixed modifications. Trypsin was selected as the enzyme, with one potential missed cleavage, and the normal cleavage mode was used. Only one search round was used with selection of "turbo" scoring. The peptide p value was 10^-2 for LTQ-OT data. Protein and peptide scores were set up to maintain the false positive rate below 5%.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2013-07-29</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD000284-2&outputMode=XML&test=no' target="_blank">Submission_2013-07-29_06:48:41.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Annarita Farina</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Homo sapiens (Human);  NCBI TaxID: 9606; </td></tr><tr><td><b>ModificationList</b></td><td>iTRAQ4plex-116 reporter+balance reagent acylated residue; iodoacetamide derivatized residue</td></tr><tr><td><b>Instrument</b></td><td>LTQ Orbitrap</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2013-06-11 08:12:24</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000284-1&test=no">1</a></td><td >2013-06-20 00:52:14</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000284-1&test=no">1</a></td><td >2014-07-25 01:15:33</td><td >announced</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD000284-2&test=no"><span class='current'>&#9205;</span> 2</a></td><td class="dataset-currentrev">2013-07-29 06:48:41</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Add reference</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Farina A, Dumonceau JM, Antinori P, Annessi-Ramseyer I, Frossard JL, Hochstrasser DF, Delhaye M, Lescuyer P, Bile carcinoembryonic cell adhesion molecule 6 (CEAM6) as a biomarker of malignant biliary stenoses. Biochim Biophys Acta, 1844(5):1018-25(2014) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=23806607" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: Human, Bile, iTRAQ, OFFGEL</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Annarita Farina</th></tr>
<tr><td>contact affiliation</td><td>Department of Human Protein Science</td></tr><tr><td>contact email</td><td>annarita.farina@unige.ch</td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/2013/06/PXD000284 target="_blank">Dataset FTP location</a></td></tr></table>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>