<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD000386</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD000386</h1>
    
	<div class="dataset-content">
	PXD000386 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Extracellular Matrix Proteome</td></tr><tr><td><b>Description</b></td><td class='breakwords'>A major obstacle in defining the exact role of extracellular matrix (ECM) in stem cell niches is the lack of suitable methods that recapitulate complex ECM microenvironments /in vitro/. Here we describe a methodology that permits reliable anchorage of native cell-secreted ECM to culture carriers. We validate our approach by fabricating two distinct types of human bone marrow (BM) specific ECM substrates that are sufficiently robust to serve as support for human mesenchymal (MSC) and hematopoietic stem and progenitor cells (HSPC) /in vitro/. The MSC-derived ECM preparations were thoroughly characterised to identify their molecular composition, structural features and nano-mechanical properties. We demonstrate the unique ability of these complex ECMs to support expansion and differentiation of BM stem cells. The established methodology enables deciphering and modulating native-like multicomponent ECMs of tissue-resident stem cells, and will therefore set the ground for a more rational design of engineered stem cell niches. Mass spectrometry: SDS-PAGE separated extracts of aaECM and osteoECM were cut into 17-20 slices according to major bands present in both lanes. LC-MS analysis was performed on an Ultimate 3000 HPLC system in front of a LTQ Orbitrap XL mass spectrometer. Top 8 peaks in mass spectra (Orbitrap, resolution 60000) were selected for fragmentation (CID, normalized collision energy 35%, activation time 30ms, q-value 0.25). Dynamic exclusion was enabled (repeat count 2, repeat duration 10s, exclusion duration 20s). MS/MS-spectra were acquired in the LTQ in centroid mode. Data processing: Proteins were identified using the MaxQuant software package version 1.2.0.18 (MPI for Biochemistry Munich, Germany) and Uniprot data base version 02/2011. Carbamidomethylation of cysteine was chosen as a fixed modification, acetylation of the N-terminus, deamidation of asparagine and oxidation of methionine as variable modifications. Proteins with at least one unique and two razor peptides were kept when present in at least two of three replicates, and gene ontologies were added using Perseus version 1.2.0.13. For selected gene ontologies LFQ-Intensities of matching proteins were summed and related to the total sum of LFQ-Intensities of all detected proteins.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2013-09-12</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD000386-2&outputMode=XML&test=no' target="_blank">Submission_2013-09-12_05:49:54.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Christian Niehage</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Homo sapiens (Human);  NCBI TaxID: 9606; </td></tr><tr><td><b>ModificationList</b></td><td>acetylated residue; iodoacetamide derivatized residue; monohydroxylated residue; deaminated residue</td></tr><tr><td><b>Instrument</b></td><td>LTQ Orbitrap</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2013-07-31 04:25:04</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000386-1&test=no">1</a></td><td >2013-09-12 04:40:41</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000386-1&test=no">1</a></td><td >2014-07-25 01:27:28</td><td >announced</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD000386-2&test=no"><span class='current'>&#9205;</span> 2</a></td><td class="dataset-currentrev">2013-09-12 05:49:54</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Add reference</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Prewitz MC, Seib FP, von Bonin M, Friedrichs J, Stißel A, Niehage C, Müller K, Anastassiadis K, Waskow C, Hoflack B, Bornhäuser M, Werner C, Tightly anchored tissue-mimetic matrices as instructive stem cell microenvironments. Nat Methods, 10(8):788-94(2013) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=23793238" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: Extracellular Matrix of Stem cells</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Christian Niehage</th></tr>
<tr><td>contact affiliation</td><td>Hoflack Lab</td></tr><tr><td>contact email</td><td>christian.niehage@biotec.tu-dresden.de</td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/2013/09/PXD000386 target="_blank">Dataset FTP location</a></td></tr></table>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>