<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD000187</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD000187</h1>
    
	<div class="dataset-content">
	PXD000187 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Extra-cellular proteins from Lactic Acid Bacteria</td></tr><tr><td><b>Description</b></td><td class='breakwords'>This data displays both known and unknown extra-cellular proteins from 13 species of Lactic Acid bacteria found in the honey-crop of the honeybee Apis. mellifera mellifera. The tryptic peptides from the secreted proteins were run on an Agilent HPLC on a C18 reverse phase column (75 µm x 150 mm, particle size 3 µm). Total run time was 90 min and flow rate 300 nl/min. Buffers used for gradient was 0.1% formic acid in water (buffer A) and 0.1% formic acid in acetonitrile (buffer B). The buffer mixing was 5 min 5% buffer B, followed by 5%-45% buffer B in a linear gradient for 50 min, followed by 45%-80% buffer B in a linear gradient for 5 min. The 80% of buffer B was then kept for 15 min and then rapidly back to 5% buffer B for the final 15 min. The fractions from HPLC were loaded on an LCQ Deca XP Plus Ion trap mass spectrometer (ThermoScientific). Genomic DNA were prepared from all 13 LAB strains depicted earlier and sequenced at MWG Eurofins Operon (Ebensburg, Germany) using Roche GS FLX Titanium technology from Roche (Basel, Switzerland).  For each genome a shotgun library was constructed with up to 700,000 reads per segment and was generated by sequencing in 2x half segment of a full FLX+ run. Each genome had an 8 kpb long-paired end library constructed. Approximately 300,000 true paired end reads, sequence tags, and scaffolds with GS FLX+ chemistry using 2x half segment of a full run were generated. Clonal amplification was performed by emPCR in both library types. The sequencing was continued until 15-20 fold coverage was reached. The obtained reads were assembled by the software Newbler 2.6 from Roche (Basel, Switzerland). ORF prediction and automated annotation was performed at Integrated Genomics Assets Inc. (Mount Prospect, Illinois, USA).  In ORF prediction three different software were used, GLIMMER, Critica, and Prokpeg. Automated annotation was performed with the ERGOTM algorithms (Integrated Genomics Assets Inc. Mount Prospect, Illinois, USA). The resulting mass spectra-files obtained from the mass spectrometry analysis were searched using MASCOT against a local database containing the predicted proteome of the 13 LAB. We used a cut off Ions score of 38 as a value for determining that the protein was identified. Individual ion scores that were greater than 38 indicated identity or extensive homology (p<0.05) of the protein. Protein sequence similarity searches were performed with software BLASTP in the software package BLAST 2.27+ against a non-redundant protein database at NCBI. Pfam (default database), and InterProScan (default databases). Expressed proteins identified by peptide mass fingerprinting were manually re-annotated.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2014-07-28</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD000187-2&outputMode=XML&test=no' target="_blank">Submission_2014-08-08_02:10:19.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="http://dx.doi.org/10.6019/PXD000187" target="_blank">http://dx.doi.org/10.6019/PXD000187</a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Supported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Eile Butler</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Firmicutes (gram-positive bacteria);  NCBI TaxID: 1239; </td></tr><tr><td><b>ModificationList</b></td><td>monohydroxylated residue; iodoacetamide derivatized residue</td></tr><tr><td><b>Instrument</b></td><td>instrument model: IonTrap XCT Ultra</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2013-03-20 03:44:14</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000187-1&test=no">1</a></td><td >2013-10-01 05:47:41</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000187-1&test=no">1</a></td><td >2014-07-28 00:31:30</td><td >announced</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD000187-2&test=no"><span class='current'>&#9205;</span> 2</a></td><td class="dataset-currentrev">2014-08-08 02:10:20</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Updated publication reference for PubMed record(s): 24148670.</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Butler È, Alsterfjord M, Olofsson TC, Karlsson C, Malmström J, Vásquez A, Proteins of novel lactic acid bacteria from Apis mellifera mellifera: an insight into the production of known extra-cellular proteins during microbial stress. BMC Microbiol, 13():235(2013) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=24148670" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: Proteins, Lactobacillus, Bifidobacterium, Mass Spec</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Eile Butler</th></tr>
<tr><td>contact affiliation</td><td>Medical Microbiology</td></tr><tr><td>contact email</td><td>eile.butler@med.lu.se</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2013/10/PXD000187 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD000187 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD000187" target="_blank">PXD000187</a><ol><li>Label: PRIDE project</li><li>Name: Extra-cellular proteins from Lactic Acid Bacteria</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>