<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD024753</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD024753</h1>
    
	<div class="dataset-content">
	PXD024753 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>LC/MS/MS Analyses of enriched ECM-enriched preparations from planaria</td></tr><tr><td><b>Description</b></td><td class='breakwords'>ECM isolation - Whole-mount planarian decellularization has been optimized based on a previous publication (Sonpho et al. 2020). We optimized three protocols for decellularization, which we refer to as No Pretreatment ECM (NP-ECM), Formaldehyde ECM (FA-ECM), and N-Acetyl Cysteine pretreatment (NAC-ECM). All worms were collected from static culture. All three protocols were designed for 20 planarians. 
Proteomics analysis - All decellularized samples were centrifuged at 16,900 g at 4C to recover the ECM pellets. Non-decellularized planarians (whole animal, WA) were prepared for comparison by freezing them in liquid nitrogen. ECM and WA pellets were resuspended in 120 uL of 100 mM Tris-HCl, pH 8.5, and 8 M Urea. The samples were vortexed vigorously to dissolve pellets. To reduce disulfide bonds, Tris(2-carboxyethyl)phosphine hydrochloride (TCEP, Pierce) was added to 5 mM and incubated at RT for 30 min. Reduced cysteines were alkylated by adding 2.4 uL of a freshly made 0.5 M chloroacetamide stock solution (Sigma) and incubating at RT in the dark for 30 min. Samples were deglycosylated with PNGase F by adding 5ul of a solution consisting of 18uL of Glycobuffer 2 (10x) and 27uL of PNGaseF (NEB, P0708S).  Endoproteinase Lys-C (Promega), 5 uL at 1 ug/uL, was added and samples were incubated at 37C overnight while shaking. Solutions were diluted to 2 M urea by adding 360 uL of 100 mM Tris-HCl and 2 uL of CaCl2. For trypsin digestion, 10 uL of Trypsin (Promega) at 0.1 ug/uL was added and samples were incubated at 37C overnight while shaking. Reactions were quenched by adding 90% formic acid to a final concentration of 5%. 
Peptides were diluted with Buffer A (5% acetonitrile (ACN), 0.1% formic acid (FA)). Then, samples were injected on an in-house packed 50 um inner diameter microcapillary column packed with 10 cm of 1.9 um Aqua C18 resin (Dr. Maisch GmbH). The samples were injected in 100% Buffer A and subsequently eluted using an Ultimate 3000 UPLC (Dionex) at a flow rate of 0.120 uL/min for 240 min from 10-40% Buffer B (80% ACN, 0.1% FA) before ramping to 95% B in 25 min. The high organic concentration was maintained for 15 min before re-equilibration and the reinjection with the next sample. Peptides were ionized by the application of a 2.5 kV positive voltage applied distally to the peptides eluting into a Q-Exactive Plus (QE+) mass spectrometer (Thermo Scientific, Waltham, MA). Full MS spectra were recorded on the eluting peptides over a 350 to 1700 m/z range at 70,000 resolving power. The AGC (automated gain control) target was 1x106 with a max injection time set to 50 ms. The top 15 peptides with charges 2-5 were fragmented by High energy Collision Dissociation (HCD) at 27% normalized collision energy (NCE) and the MS/MS fragmentation was collected at 17,500 resolving power, with an AGC target of 1x105 and a maximum injection time set to 150 ms. Dynamic exclusion was enabled for 30 seconds. Mass spectrometer scan functions and HPLC solvent gradients were controlled by the XCalibur data system (Thermo Scientific, Waltham, MA).
MS/MS spectra were first searched using ProLuCID v. 1.3.3 with a peptide mass tolerance of 10 ppm and 25 ppm for peptide and fragment ions, respectively. Trypsin specificity was imposed on both ends of candidate peptides during the search against a protein database containing 30,863 S. mediterranea predicted protein sequences as well as 483 usual contaminants such as human keratins, IgGs and proteolytic enzymes. To estimate false discovery rates (FDR), each protein sequence was randomized (keeping the same amino acid composition and length) and the resulting "shuffled" sequences were added to the database, for a total search space of 62,564 amino acid sequences. A mass shift of 57.0215 Da was statically added to cysteine residues to account for alkylation by CAM, mass shifts of +15.9949 was differentially added to methionine, lysine, and proline residues to account for oxidation and hydroxylation, and -0.98401 Da was differentially added to asparagine residues to account for deglycosylation by PNGaseF. DTASelect v.1.9 was used in combination with an in-house software, swallow v1.0 to select and sort peptide/spectrum matches (PSMs) to FDRs of less than 1% at the peptide and protein levels. 
</td></tr><tr><td><b>HostingRepository</b></td><td>MassIVE</td></tr><tr><td><b>AnnounceDate</b></td><td>2021-08-23</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD024753.0-1&outputMode=XML&test=no' target="_blank">Submission_2021-08-23_12:25:28.688.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Non peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Supported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Stowers Proteomics</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Schmidtea mediterranea;  common name: freshwater planarian;  NCBI TaxID: 79327; </td></tr><tr><td><b>ModificationList</b></td><td>S-carboxamidomethyl-L-cysteine; L-methionine sulfoxide; monohydroxylated lysine; monohydroxylated proline; L-asparagine amide</td></tr><tr><td><b>Instrument</b></td><td>Q Exactive Plus</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2021-03-15 16:09:21</td><td >ID requested</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD024753-1&test=no"><span class='current'>&#9205;</span> 1</a></td><td class="dataset-currentrev">2021-08-23 12:25:29</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev"></td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>no publication</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: Extracellular matrix</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Laurence Florens</th></tr>
<tr><td>contact affiliation</td><td>The Stowers Institute for Medical Research</td></tr><tr><td>contact email</td><td>laf@stowers.org</td></tr><tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Stowers Proteomics</th></tr>
<tr><td>contact affiliation</td><td>Stowers Institute for Med Res</td></tr><tr><td>contact email</td><td>proteomicslab@stowers.org</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=http://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=3ca7d76fe31a4400be124e0d5cf02202 target="_blank">MassIVE dataset URI</a></td></tr><tr><td><a href=ftp://massive.ucsd.edu/MSV000087054/ target="_blank">Dataset FTP location</a></td></tr></table>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>