<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD016497</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD016497</h1>
    
	<div class="dataset-content">
	PXD016497 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>H3 N-terminal tail peptide interactome</td></tr><tr><td><b>Description</b></td><td class='breakwords'>The closely related replicative H3 and non-replicative H3.3 variants show specific requirement during development in vertebrates. Whether it involves distinct mode of deposition or unique roles once incorporated into chromatin remains unclear. To disentangle the two aspects, we took advantage of the Xenopus early development combined with chromatin assays. We systematically mutated H3.3 at each four residues that differ from H3.2 and tested their ability to rescue developmental defects due to endogenous H3.3 depletion. Surprisingly, all H3.3 mutated variants functionally complemented endogenous H3.3, regardless of their incorporation pathways, except for one residue, the serine at position 31. The phosphorylation at this unique residue occurs onto chromatin with a peak in late mitosis, and depends on the networks of cell cycle kinases. Notably, while the alanine substitution failed to rescue H3.3 depletion, a phosphomimic residue sufficed. Based on proteomics studies with histone peptides and Xenopus extracts, we find that the phosphomimic histone mutant attracts transcription related factors. Furthermore, we evidence a crosstalk whereby phosphorylation on H3.3S31 favors H3.3K27ac. At gastrulation, we conclude that the critical importance of the H3.3S31 residue is independent of the variant incorporation pathway. It rather reflects a signaling role engaging key binding partners and crosstalks on neighboring amino acids. We discuss how this single evolutionary conserved residue conveys both in interphase and mitosis unique properties for this variant in vertebrates during cell cycle and cell fate commitment.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2020-03-13</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD016497.0-1&outputMode=XML&test=no' target="_blank">Submission_2020-03-13_00:24:29.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Valentin SABATET</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Xenopus laevis (African clawed frog);  NCBI TaxID: 8355; </td></tr><tr><td><b>ModificationList</b></td><td>No PTMs are included in the dataset</td></tr><tr><td><b>Instrument</b></td><td>Orbitrap Fusion</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2019-11-28 03:17:28</td><td >ID requested</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD016497-1&test=no"><span class='current'>&#9205;</span> 1</a></td><td class="dataset-currentrev">2020-03-13 00:24:30</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev"></td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Sitbon D, Boyarchuk E, Dingli F, Loew D, Almouzni G, Histone variant H3.3 residue S31 is essential for Xenopus gastrulation regardless of the deposition pathway. Nat Commun, 11(1):1256(2020) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=32152320" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: Histone variant, H3.3S31, Xenopus laevis, development</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Damarys Loew</th></tr>
<tr><td>contact affiliation</td><td>Head of the Curie Institute Mass Spectrometry Platform</td></tr><tr><td>contact email</td><td>damarys.loew@curie.fr</td></tr><tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Valentin SABATET</th></tr>
<tr><td>contact affiliation</td><td>Curie Institute</td></tr><tr><td>contact email</td><td>valentin.sabatet@curie.fr</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2020/03/PXD016497 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD016497 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD016497" target="_blank">PXD016497</a><ol><li>Label: PRIDE project</li><li>Name: H3 N-terminal tail peptide interactome</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>