<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD000380</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD000380</h1>
    
	<div class="dataset-content">
	PXD000380 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>KMH2 control treatment</td></tr><tr><td><b>Description</b></td><td class='breakwords'>We investigate molecular effects of glucosamine supplements, a popular safe alternative to non-steroidal anti-inflamatory drugs for decreasing pain, inflammation and keeping joints healthy. Numerous publications state an array of different molecular effects upon glucosamine treatment. We question whether these differences resulted mainly from focusing on a specific sub proteome or because different cell lines or tissues were used in the previous studies. To address this question global MS- and transcription array-based glucosamine drug profiling were performed on lymphocyte derived cell lines. We combined global label free MS-based protein quantitation with an open search strategy to obtain the best possible proteome coverage. Our result in large part support previous findings in a variety of cell models. Glucosamine induce O-GlcNAcylation but we also observe global changes in acetylation, methylation, phosphorylation. Furthermore, we mapped novel O-GlcNAc sites, on ER lumen proteins, regulated upon glucosamine treatment which provides novel insights into the role of glucosamine in ER stress. Preprocessing of MS data: All Q-Exactive data were calibrated using polycyclodi-methylsiloxane (PCMs—outgassed material from semiconductors) present in the ambient air and Bis(2-Ethylhexyl)(Phthalate) (DEHP—from plastic)1, 2 using both MaxQuant version 1.3.0.53 and modular VEMS mVEMS4. mVEMS4 further allows alternative parent ion annotations for each MS/MS spectrum which is needed if two peptide elution profiles overlap in the m/z and retention time dimension. By allowing alternative parent ion annotation for each MS/MS spectrum, provides a space efficient data format. Furthermore these alternative parent ion annotations were taken into account during the search. Database dependent search of MS data: All data were searched with MaxQuant version 1.3.0.53 and VEMS5. Mass accuracy was set to 10 ppm for peptides and 10 mDa for peptide fragments. Four missed cleavages were specified and the database UniProtKB/TrEMBL were used including permutated protein sequences, leaving Lys and Arg in place, together with common contaminants such as human keratins, bovine serum proteins and proteases6. Fixed modification of carbamidomethyl cysteine was included in the search parameters. In MaxQuant the following variable modifications were considered: Met-loss (UNIMOD 765), Oxidized methionine (UNIMOD 885), O-GlcNAc (HexNAc, UNIMOD 43). In VEMS a list of variable 36 modifications were considered for all data against the full protein database were performed. Protein N-terminal Met-loss is not in the list of VEMS modifications since VEMS by default always looks for N-terminal Met-loss. The modifications Phosphate-ribosylation (UNIMOD 1356), ADP ribose (UNIMOD 213), hydroxylation (UNIMOD 35) and Myristoylation (UNIMOD 45) were not frequent at a 1% FDR threshold and the search was therefore repeated excluding these modifications.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2014-10-23</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD000380-2&outputMode=XML&test=no' target="_blank">Submission_2014-10-23_02:43:27.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Rune Matthiesen</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Homo sapiens (Human);  NCBI TaxID: 9606; </td></tr><tr><td><b>ModificationList</b></td><td>deamidated residue; monohydroxylated residue; acetylated residue; iodoacetamide derivatized residue; phosphorylated residue; carbamoylated residue; monomethylated residue</td></tr><tr><td><b>Instrument</b></td><td>Q Exactive</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2013-07-30 01:18:16</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000380-1&test=no">1</a></td><td >2014-08-20 02:48:25</td><td >announced</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD000380-2&test=no"><span class='current'>&#9205;</span> 2</a></td><td class="dataset-currentrev">2014-10-23 02:43:28</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Updated project metadata.</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Carvalho AS, Ribeiro H, Voabil P, Penque D, Jensen ON, Molina H, Matthiesen R, Global mass spectrometry and transcriptomics array based drug profiling provides novel insight into glucosamine induced endoplasmic reticulum stress. Mol Cell Proteomics, 13(12):3294-307(2014) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=25128556" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>curator keyword: Biomedical</td></tr><tr><td>submitter keyword: KMH2 PTMs drug profilling</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Rune Matthiesen</th></tr>
<tr><td>contact affiliation</td><td>Population genetics</td></tr><tr><td>contact email</td><td>rmatthiesen@ipatimup.pt</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2014/08/PXD000380 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD000380 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD000380" target="_blank">PXD000380</a><ol><li>Label: PRIDE project</li><li>Name: KMH2 control treatment</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>