<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD023129</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD023129</h1>
    
	<div class="dataset-content">
	PXD023129 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>MudPIT analysis of proteins extracted from mouse KH2ES cell nuclei and enriched by binding to immobilized DNA fragments</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Sample Preparation: To identify proteins that are recruited by Hoxb1 to two newly identified DNA motifs, namely Motif_2 and LONG2x, nuclear extracts from 24hrs Retinoic acid- and Doxycycline-induced KH2 cell line with inducible Hoxb1 were added to DNA fragments immobilized on magnetic beads. Five experimental conditions were used: motif_2 (15bp) DNA, long2X (45bp) DNA, Hoxb1-ARE DNA as positive control, and 2 negative controls, random DNA and beads-only. In all cases, proteins were incubated at 30C for 30min to allow binding, followed by 2 sequential washes with buffer containing 0.1mg/ml BSA. Proteins were eluted by 2% SDS, 50mM Tris-HCl pH8.8 at 70C for 10 min, then precipitated by 20% TCA. 

MudPIT Analysis
TCA-precipitated proteins were urea-denatured, reduced, alkylated and digested with endoproteinase Lys-C (Roche) followed by modified trypsin (Promega). Peptide mixtures were loaded onto 250 um fused silica microcapillary columns packed with strong cation exchange resin (Luna, Phenomenex) and 5-um C18 reverse phase (Aqua, Phenomenex), and then connected to a 100 um fused silica microcapillary column packed with 5-um C18 reverse phase (Aqua, Phenomenex). Loaded microcapillary columns were placed in-line with a Quaternary Agilent 1100 series HPLC pump and a LTQ linear ion trap mass spectrometer equipped with a nano-LC electrospray ionization source (ThermoScientific, San Jose, CA). Fully automated 10-step MudPIT runs were carried out on the electrosprayed peptides. Tandem mass (MS/MS) spectra were interpreted using ProluCID (v. 1.3.3) against a database consisting of 61441 non-redundant M. musculus proteins (NCBI, 2020-11-03 release), 426 usual contaminants (human keratins, IgGs, and proteolytic enzymes). To estimate false discovery rates (FDR)s, the amino acid sequence of each non-redundant protein entry was randomized, resulting in a total search space of 123728 non-redundant sequences. All cysteines were considered as fully carboxamidomethylated (+57 Da statically added), while methionine oxidation was searched as a differential modification.  DTASelect (v. 1.9) and swallow, an in-house developed software, were used to filter ProLuCID search results at given FDRs at the spectrum, peptide, and protein levels.  Here all controlled FDRs were less than 5%.  All 16 data sets were contrasted against their merged data set, respectively, using Contrast v 1.9 and in house developed sandmartin v0.0.1.  Our in-house developed software, NSAF7 v0.0.1, was used to generate spectral count-based label free quantitation results.</td></tr><tr><td><b>HostingRepository</b></td><td>MassIVE</td></tr><tr><td><b>AnnounceDate</b></td><td>2021-01-28</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD023129.0-1&outputMode=XML&test=no' target="_blank">Submission_2021-01-28_12:31:50.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Non peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Supported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Laurence Florens</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Mus musculus;  common name: house mouse;  NCBI TaxID: 10090; </td></tr><tr><td><b>ModificationList</b></td><td>S-carboxamidomethyl-L-cysteine; L-methionine sulfoxide</td></tr><tr><td><b>Instrument</b></td><td>LTQ</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2020-12-14 15:06:32</td><td >ID requested</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD023129-1&test=no"><span class='current'>&#9205;</span> 1</a></td><td class="dataset-currentrev">2021-01-28 12:31:50</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev"></td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>no publication</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: KH2 embryonic stem cells, Hoxb1, DNA binding</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Laurence Florens</th></tr>
<tr><td>contact affiliation</td><td>The Stowers Institute for Medical Research</td></tr><tr><td>contact email</td><td>laf@stowers.org</td></tr><tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Laurence Florens</th></tr>
<tr><td>contact affiliation</td><td>Stowers Institute for Medical Research</td></tr><tr><td>contact email</td><td>laf@stowers.org</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=http://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=745c5ef411da4f82b96411b05252c70e target="_blank">MassIVE dataset URI</a></td></tr><tr><td><a href=ftp://massive.ucsd.edu/MSV000086596/ target="_blank">Dataset FTP location</a></td></tr></table>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>