<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD000133</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD000133</h1>
    
	<div class="dataset-content">
	PXD000133 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Equine Cerebrospinal Fluid</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Cerebrospinal fluid is a diagnostic biofluid that is reflective of the overall health of the patient.  Proteomic profiling of human CSF has been performed on a variety of disease states, but a comprehensive proteomic profile of equine CSF has not been completed until now.  A total of 320 proteins were confidently identified in a pooled sample representative of 6 healthy horses.  Gene Ontology terms were mapped in Uniprot, and normalized spectral abundance factors were calculated as a measure of relative abundance. Briefly, CSF was collected from healthy horses via subarachnoid catheter or manual draw, protease inhibitors were added (Pierce, Rockford, IL), and samples were frozen at -80oC (Figure 1).  Protein concentrations were determined via Bradford Assay  (Thermo Scientific, Rockford, IL) and 30 ug of each sample underwent in-solution digestion using ProteaseMAX (Promega, Madison, WI) and urea.  Samples were solubilized in 8 M urea, 0.2% protease max, then reduced, alkylated, and digested with 1% protease max and trypsin at 37oC for 3 hours.  Samples were dried in a Speed VacÂ® vacuum centrifuge, desalted using Pierce PepClean C18 spin columns (Pierce, Rockford, IL), dried and resuspended in 30 ul 3% ACN, 0.1% formic acid.    All solvents, water, and acid were LC-MS/MS grade from Sigma (St. Louis, MO).  Online 2-dimensional LC-MS/MS with SCX (strong cation exchange) and subsequent reverse phase chromatography was performed as follows.  10 ug of digested peptides from a sample were loaded onto a Zorbax BIO-SCX II 3.5 umm, 50 x 0.8 mm column (Agilent Technologies, Santa Clara, CA).  Peptides were eluted off of the SCX column step-wise using increasing concentrations of NaCl in 0.3% ACN, 0.1% FA (20 ul NaCl salt injections:  15, 30, 45, 60, 75, 90, 120, 150, 300, 500 mM).  Peptides from each individual salt injection were then purified and concentrated using an on-line enrichment column (Agilent Zorbax C18, 5 um, 5 x 0.3mm).  Subsequent chromatographic separation was performed on a reverse phase nanospray column (Agilent 1100 nanoHPLC, Zorbax C18, 5um, 75 um ID x 150mm column) using a 60 minute linear gradient from 25%-55% buffer B (90% ACN, 0.1% formic acid) at a flow rate of 300 nanoliters/min.   Peptides were eluted directly into the mass spectrometer (Thermo Scientific LTQ linear ion trap) and spectra were collected over a range of 200-2000 m/z using a dynamic exclusion limit of 2 MS/MS spectra of a given peptide mass for 30 s (exclusion duration of 90 s).  Compound lists of the resulting spectra were generated using Bioworks 3.0 software (Thermo Scientific) with an intensity threshold of 5,000 and 1 scan/group.  This workflow generates 10 raw data files per sample. MS/MS spectra were searched against the NCBInr equine database concatenated to a reverse database (14,473 entries) using the Mascot database search engine (Matrix Science, version 2.3.02) and SEQUEST (version v.27, rev. 11, Sorcerer, Sage-N Research).  The following search parameters were used: average mass, peptide mass tolerance of 2.5 Da, fragment ion mass tolerance of 1.0 Da, complete tryptic digestion allowing one missed cleavage, variable modification of methionine oxidation, and a fixed modification of cysteine carbamidomethylation. Peptide identifications from both of the search engines were combined using protein identification algorithms in Scaffold 3 (Version 3.6.4, Proteome Software, Portland, OR).  All data files were then combined using the MudPIT option in Scaffold 3 generating a composite listing for all proteins identified across all runs.  Thresholds were set to 99% and 95% protein and peptide probability respectively, and a 2 unique peptide minimum was required.  The peptide false discovery rate (FDR) was less than 0.2% after manual validation of all proteins identified by 2 unique peptides.  Criteria for manual validation included the following: 1) a minimum of at least 3 theoretical y or b ions in consecutive order that are peaks greater than 5% of the maximum intensity; 2) an absence of prominent unassigned peaks greater than 5% of the maximum intensity; and 3) indicative residue specific fragmentation, such as intense ions N-terminal to proline and immediately C-terminal to aspartate and glutamate.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2014-07-25</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD000133-2&outputMode=XML&test=no' target="_blank">Submission_2014-07-25_00:50:59.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Carolyn Broccardo</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Equus caballus (Horse);  NCBI TaxID: 9796; </td></tr><tr><td><b>ModificationList</b></td><td>No PTMs are included in the dataset</td></tr><tr><td><b>Instrument</b></td><td>LTQ</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2013-01-28 03:29:17</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000133-1&test=no">1</a></td><td >2014-04-23 03:41:46</td><td >announced</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD000133-2&test=no"><span class='current'>&#9205;</span> 2</a></td><td class="dataset-currentrev">2014-07-25 00:51:00</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Updated project metadata.</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Dataset with its publication pending</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: equine, CSF, proteomics</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Carolyn Broccardo</th></tr>
<tr><td>contact affiliation</td><td>Proteomics and Metabolomics Facility</td></tr><tr><td>contact email</td><td>carolyn.broccardo@colostate.edu</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2014/04/PXD000133 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD000133 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD000133" target="_blank">PXD000133</a><ol><li>Label: PRIDE project</li><li>Name: Equine Cerebrospinal Fluid</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>