<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD014240</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD014240</h1>
    
	<div class="dataset-content">
	PXD014240 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Combination of proteogenomics with peptide de novo sequencing identifies new genes and hidden posttranscriptional modifications</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Proteogenomics, the combination of proteomics, genomics and transcriptomics, has considerably improved genome annotation in under-studied phylogenetic groups, where homology information is missing. Yet, it can also be advantageous when re-investigating well-annotated genomes. Here, we apply an advanced proteogenomics approach, combining standard proteogenomics with peptide de novo sequencing, to refine annotation of the well-studied model fungus Sordaria macrospora. We investigated samples from different developmental and physiological conditions, resulting in detection of 104 hidden proteins and annotation changes in 575 genes, including  389  splice site refinements. Significantly, our approach provides peptide-level evidence for 113 single amino acid variations and 15 C-terminal protein elongations originating from A-to-I RNA editing, a phenomenon recently detected in fungi. Co-expression and phylostratigraphic analysis of the refined proteome suggests new functions in evolutionary young genes correlated with distinct developmental stages. In conclusion, our advanced proteogenomics approach is highly supportive to promote functional studies of model systems.  Proteogenomics, the combination of proteomics, genomics and transcriptomics, has considerably improved genome annotation in under-studied phylogenetic groups, where homology information is missing. Yet, it can also be advantageous when re-investigating well-annotated genomes. Here, we apply an advanced proteogenomics approach, combining standard proteogenomics with peptide de novo sequencing, to refine annotation of the well-studied model fungus Sordaria macrospora. We investigated samples from different developmental and physiological conditions, resulting in detection of 104 hidden proteins and annotation changes in 575 genes, including  389  splice site refinements. Significantly, our approach provides peptide-level evidence for 113 single amino acid variations and 15 C-terminal protein elongations originating from A-to-I RNA editing, a phenomenon recently detected in fungi. Co-expression and phylostratigraphic analysis of the refined proteome suggests new functions in evolutionary young genes correlated with distinct developmental stages. In conclusion, our advanced proteogenomics approach is highly supportive to promote functional studies of model systems.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2019-09-25</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD014240.0-2&outputMode=XML&test=no' target="_blank">Submission_2019-10-25_05:59:17.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Bernhard Blank-Landeshammer</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Sordaria macrospora;  NCBI TaxID: 5147; </td></tr><tr><td><b>ModificationList</b></td><td>monohydroxylated residue; iodoacetamide derivatized residue</td></tr><tr><td><b>Instrument</b></td><td>Orbitrap Fusion Lumos; Q Exactive</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2019-06-13 00:23:49</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD014240-1&test=no">1</a></td><td >2019-09-25 05:27:35</td><td >announced</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD014240-2&test=no"><span class='current'>&#9205;</span> 2</a></td><td class="dataset-currentrev">2019-10-25 05:59:19</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">2019-10-25: Updated publication reference for PubMed record(s): 31615963.</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Blank-Landeshammer B, Teichert I, Märker R, Nowrousian M, Kück U, Sickmann A,  Sequencing Identifies New Genes and Hidden Posttranscriptional Modifications. mBio, 10(5):(2019) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=31615963" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>curator keyword: Technical, Biological</td></tr><tr><td>submitter keyword: Sordaria macrospora, LC-MSMS, de novo peptide sequencing, Proteogenomics, RNA Editing</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Albert Sickmann</th></tr>
<tr><td>contact affiliation</td><td>Leibniz-Institut für Analytische Wissenschaften – ISAS – e.V., Otto-Hahn-Str. 6b, 44227 Dortmund, Germany</td></tr><tr><td>contact email</td><td>sickmann@isas.de</td></tr><tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Bernhard Blank-Landeshammer</th></tr>
<tr><td>contact affiliation</td><td>Leibniz-Institut für Analytische Wissenschaften – ISAS – e.V., Dortmund</td></tr><tr><td>contact email</td><td>bernhard.blank@isas.de</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2019/09/PXD014240 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD014240 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD014240" target="_blank">PXD014240</a><ol><li>Label: PRIDE project</li><li>Name: Combination of proteogenomics with peptide de novo sequencing identifies new genes and hidden posttranscriptional modifications</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>