<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD000141</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD000141</h1>
    
	<div class="dataset-content">
	PXD000141 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Mycobacterium tuberculosis ATP-binding Proteome</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Mtb was grown as either an actively growing normal culture or as a hypoxia-induced dormant culture. Whole cell lysates of live bacteria were obtained by mechanical disruption with silica beads. ATP-binding proteins were covalently labeled with desthiobiotin-tagged ATP (ActivX, Thermo Pierce). This chemoproteomic approach profiled the ATP-binding proteins present in either metabolic state. Labeled proteins were enriched via streptavidin affinity chromatography, digested with trypsin and subjected to tandem mass spectrometry (LC-MS/MS) for the identification of proteins containing a desthiobiotin modification of lysine (K +196 Da). Differential abundance of nucleotide binding proteins between the two growth conditions were quantified using label-free spectral counting and normalized spectral abundance factors (NSAF). DATABASE SEARCHING: Tandem mass spectra were extracted, charge state deconvoluted and deisotoped by Xcalibur version 2.2 SP1. All MS/MS samples were analyzed using Mascot (Matrix Science, London, UK; version 2.3.02) and SEQUEST (Thermo Fisher Scientific, San Jose, CA, USA; version v.27, rev. 11). Mascot and SEQUEST were set up to search the MtbReverse041712 database (7992 entries) assuming the digestion enzyme Trypsin. Parameters for both search engines were set to a fragment ion mass tolerance of 1.0 Da and a parent ion tolerance of 2.5 Da.  Oxidation of methionine, iodoacetamide derivative of cysteine and the desthiobiotin modification of lysine were specified in Mascot and SEQUEST as variable modifications. PROTEIN IDENTIFICATION AND LABEL-FREE QUANTITATION-- Scaffold (version Scaffold_3.6.1, Proteome Software Inc., Portland, OR) was used to validate MS/MS based peptide and protein identifications. Peptide identifications were accepted if they exceeded specific database search engine thresholds. Mascot identifications required ion scores to be greater than the associated identity scores and 50, 65, 65 and 65 for singly, doubly, triply and quadruply charged peptides. SEQUEST identifications required deltaCn scores of greater than 0.2 and XCorr scores of greater than 1.8, 2.0, 3.0 and 4.0 for singly, doubly, triply and quadruply charged peptides. Proteins identifications were accepted if they contained at least one identified peptide in at least two biological replicates. Peptide spectra meeting the most minimum requirement were manually inspected for quality. Quantification of proteins was performed on normalized spectral abundance factors for each protein (NSAF). BLAST-BASED SEQUENCE DESCRIPTION:  The most relevant description for each of the sequences was acquired based on the significant BLAST results.  The homologs for the sequences were retrieved using the Blastp algorithm and the nonredundant database of NCBI. The Blast2GO suite was used. GENE ONTOLOGY ANNOTATION:  The Pfam domains were mapped to Gene Ontology (GO) terms using the lookup table provided by Pfam2go (http://www.geneontology.org/external2go/pfam2go).  An in-house script was written to retrieve GO annotations based on the root term as "molecular function" and their distance from the root term. PFAM DOMAIN-BASED ANNOTATION:  The InterPro and Pfam IDs corresponding to the GO term "ATP binding" (GO ID:0005524) were retrieved using the QuickGO (www.ebi.ac.uk/QuickGO/<http://www.ebi.ac.uk/QuickGO/>) and InterPro BioMart web services (http://www.ebi.ac.uk/interpro/biomart/martview). The ATP-binding associated domains were queried against the ATP-binding proteome data sets according to spectral quality (high, medium, low confidence). Their Pfam and InterPro descriptions, were identified using the InterProscan Web service, which was accessed via the Pipeline Pilot (Accelrys) implementation in the sequence analysis collection.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2013-04-08</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD000141-2&outputMode=XML&test=no' target="_blank">Submission_2013-04-08_06:44:04.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Karen Dobos</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Mycobacterium tuberculosis H37Rv;  NCBI TaxID: 83332; </td></tr><tr><td><b>ModificationList</b></td><td>iodoacetamide derivatized residue; monohydroxylated residue; N6-crotonyl-L-lysine</td></tr><tr><td><b>Instrument</b></td><td>LTQ</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2013-02-07 02:25:55</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000141-1&test=no">1</a></td><td >2013-04-08 06:34:33</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000141-1&test=no">1</a></td><td >2014-07-25 00:52:21</td><td >announced</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD000141-2&test=no"><span class='current'>&#9205;</span> 2</a></td><td class="dataset-currentrev">2013-04-08 06:44:04</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Add reference</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Wolfe LM, Veeraraghavan U, Idicula-Thomas S, Schürer S, Wennerberg K, Reynolds R, Besra GS, Dobos KM, A chemical proteomics approach to profiling the ATP-binding proteome of Mycobacterium tuberculosis. Mol Cell Proteomics, 12(6):1644-60(2013) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=23462205" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: M. tuberculosis, LC-MS/MS, ABPP, ATP</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Karen Dobos</th></tr>
<tr><td>contact affiliation</td><td>Microbiology, Immunology, and Pathology</td></tr><tr><td>contact email</td><td>kdobos@colostate.edu</td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/2013/04/PXD000141 target="_blank">Dataset FTP location</a></td></tr></table>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>