<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD000196</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD000196</h1>
    
	<div class="dataset-content">
	PXD000196 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Shotgun proteomics of nucleoids from irradiated Deinococcus cells</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Nucleoids have been purified from Deinococcus deserti and Deinococcus radiodurans cells subjected to irradiation and short recovery (six replicates for each strain). The nucleoids have been analyzed by shotgun proteomics as described in Toueille M et al (2012) J Proteomics. Comparative proteomics pointed at specific proteins recruited at the nucleoids during the recovery stage after DNA damages produced by the irradiation. Data processing and bioinformatics: Peak lists were generated with the MASCOT DAEMON software (version 2.2.2) from Matrix Science using the extract_msn.exe data import filter (ThermoFisher) from the Xcalibur FT package (version 2.0.7) from ThermoFisher.  Data import filter options were set at: 400 (minimum mass), 5000 (maximum mass), 0 (grouping tolerance), 0 (intermediate scans), and 1000 (threshold). Using the MASCOT search engine (version 2.2.04) from Matrix Science, we searched all MS/MS spectra against in-house polypeptide sequence databases. For Deinococcus radiodurans MS/MS assignments, the database contained i) the sequence of all currently annotated proteins coded by D. radiodurans BAA-816 genome (2629 proteins from chromosome 1 (NC_001263), 368 proteins from chromosome 2 (NC_001264), 130 proteins from plasmid MP1 (NC_000958), and 35 proteins from plasmid CP1 (NC_000958)), ii) 28 manual protein sequence curations from D. radiodurans R1, and iii) 121 additional ORF sequences predicted by the CONSORF consensus prediction system.This database thus comprises 3,311 polypeptide sequences, totaling 1,006,757 amino acids. For Deinococcus  deserti MS/MS assignments, the database contained the sequence of all annotated proteins coded by D. deserti VCD115 chromosome and plasmids. This database comprises 3,455 polypeptide sequences, totaling 1,083,334 amino acids. Searches for tryptic peptides were performed with the following parameters: full-trypsin specificity, a mass tolerance of 10 ppm on the parent ion and 0.5 Da on the MS/MS, static modifications of carboxyamidomethylated Cys (+57.0215), and dynamic modifications of oxidized Met (+15.9949). The maximum number of missed cleavages was set at 1. All peptide matches with a peptide score below a stringent P value of 0.001 were filtered by the IRMa 1.18.1 parser. A protein was considered validated when at least two different peptides were detected in the same experiment. False-positives rate for protein identification was estimated using the appropriate decoy database as below 0.1% with these parameters.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2014-07-28</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD000196-2&outputMode=XML&test=no' target="_blank">Submission_2014-07-28_00:32:43.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="http://dx.doi.org/10.6019/PXD000196" target="_blank">http://dx.doi.org/10.6019/PXD000196</a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Supported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Jean ARMENGAUD</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Deinococcus deserti (strain VCD115 / DSM 17065 / LMG 22923);  NCBI TaxID: 546414;  scientific name: Deinococcus radiodurans;  NCBI TaxID: 1299; </td></tr><tr><td><b>ModificationList</b></td><td>iodoacetamide - site C: 57.021464; monohydroxylated residue; monohydroxylated residue: 15.994915; iodoacetamide - site C</td></tr><tr><td><b>Instrument</b></td><td>LTQ Orbitrap; instrument model: LTQ-Orbitrap</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2013-03-28 03:54:44</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000196-1&test=no">1</a></td><td >2014-01-31 07:05:26</td><td >announced</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD000196-2&test=no"><span class='current'>&#9205;</span> 2</a></td><td class="dataset-currentrev">2014-07-28 00:32:44</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Updated project metadata.</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>de la Tour CB, Passot FM, Toueille M, Mirabella B, Gu√©rin P, Blanchard L, Servant P, de Groot A, Sommer S, Armengaud J, Comparative proteomics reveals key proteins recruited at the nucleoid of Deinococcus after irradiation-induced DNA damage. Proteomics, 13(23-24):3457-69(2013) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=24307635" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: Nucleoids, DNA repair, Deinococcus deserti, Deinococcus radiodurans</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Jean ARMENGAUD</th></tr>
<tr><td>contact affiliation</td><td>SBTN</td></tr><tr><td>contact email</td><td>jean.armengaud@cea.fr</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2014/01/PXD000196 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD000196 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD000196" target="_blank">PXD000196</a><ol><li>Label: PRIDE project</li><li>Name: Shotgun proteomics of nucleoids from irradiated Deinococcus cells</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>