<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD004957</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD004957</h1>
    
	<div class="dataset-content">
	PXD004957 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Tandem affinity purification and Multidimensional Protein Identification Technology to identify USP44 interacting proteins</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Nuclear extracts were isolated from 293T cells that stably express USP44 fused with N-terminal FLAG and HA affinity tags (FH-USP44), and then analyzed by tandem affinity purification (TAP) followed by Multidimensional Protein Identification Technology (MudPIT) to uncover USP44 associated proteins. TAP of FH-tagged CETN2 (centrin2), the only known USP44 interacting protein, was also performed followed by MudPIT analysis. A FH-vector control was purified and analyzed in parallel.
TCA-precipitated proteins were denatured in 8M urea (Sigma-Aldrich), reduced with Tris(2-Carboxylethyl)-Phosphine Hydrochloride (TCEP; Pierce) and alkylated with chloroacetamide (CAM; Sigma-Aldrich), then digested with endoproteinase LysC (Roche) followed by trypsin (Promega). The resulting peptide mixtures were pressure-loaded onto 250 um fused silica microcapillary columns packed first with 3 cm of 5-um Strong Cation Exchange (SCX) material (Luna; Phenomenex), followed by 1 cm of 5-um C18 reverse phase (Aqua; Phenomenex). Loaded 250 um columns were connected using a filtered union (UpChurch) to 100 um fused-silica columns pulled to a 5 um tip using a P 2000 CO2 laser puller (Sutter Instruments) and packed with 9 cm of reverse phase material. The loaded microcapillary columns were placed in-line with a Quaternary Agilent 1260 series HPLC. Three channels of the HPLC were filled with: 5% acetonitrile and 0.1% formic acid (channel A); 80% acetonitrile and 0.1% formic acid (channel B) and 0.5 M ammonium acetate, 5% acetonitrile, and 0.1% formic acid (channel C);. Fully automated 12 step chromatography runs were carried out with a flow rate of 0.1 mL/min, split to 250 nL/min at the tip of the column. Peptides were sequentially eluted from the SCX resin to the reverse phase resin by a 10 min-long salt step of increasing concentrations (0, 7, 15, 25, 35, 45, 60, 80, 100, 90, 85, and 100% C for steps 1 through 12, respectively), followed by the organic gradient to 75% solvent. The application of a 2.5 kV distal voltage electrosprayed the eluting peptides directly into a Velos-Pro-Orbitrap-ETD Hybrid mass spectrometer (Thermo Scientific) equipped with a custom-made nano-LC electrospray ionization source. Full MS spectra are recorded on the peptides over a 400 to 1,600 m/z range in the LTQ Velos ion trap, followed by 10 FT tandem mass (MS/MS) events in the Orbitrap at 60K resolution, sequentially generated in a data-dependent manner on the first to tenth most intense ions selected from the full MS spectrum (at 35% collision energy). Dynamic exclusion enabled for 90sec. Mass spectrometer scan functions and HPLC solvent gradients were controlled by the XCalibur data system (Thermo Scientific).
RAW files were extracted into .ms2 file format using RawDistiller v. 1.0 (in-house developed software). The MS/MS spectra were searched using SEQUEST v.27 (rev.9) with a peptide mass tolerance of 7 ppm and of +/- 0.5 amu for fragment ions. No enzyme specificity was imposed during the SEQUEST searches against a protein database containing 30499 non-redundant human proteins (NCBI 2012-08-27 release), as well as 204 proteins consisting of usual contaminants such as human keratins, IgGs and proteolytic enzymes or added affinity-tagged proteins . To estimate false discovery rates (FDR), each protein sequence was randomized (keeping the same amino acid composition and length) and the resulting &quot;shuffled&quot; sequences were added to the database used for the SEQUEST searches, for a total search space of 61406 amino acid sequences. To account for alkylation by CAM, 57.02146 Da were statically added to cysteine residues for all searches.
Peptide/spectrum matches were sorted and selected using DTASelect with the following criteria set: spectra/peptide matches were only retained if they had a DeltCn of at least 0.08, and minimum XCorr of 1.8 for singly-, 2.0 for doubly-, and 3.0 for triply-charged spectra.  In addition, peptides had to be fully-tryptic and at least 7 amino acids long.  </td></tr><tr><td><b>HostingRepository</b></td><td>MassIVE</td></tr><tr><td><b>AnnounceDate</b></td><td>2016-11-28</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD004957-1&outputMode=XML&test=no' target="_blank">Submission_2016-11-28_07:37:47.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Laurence Florens</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Homo sapiens;  common name: human;  NCBI TaxID: 9606; </td></tr><tr><td><b>ModificationList</b></td><td>S-carboxamidomethyl-L-cysteine</td></tr><tr><td><b>Instrument</b></td><td>LTQ Orbitrap Velos</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2016-09-13 07:39:42</td><td >ID requested</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD004957-1&test=no"><span class='current'>&#9205;</span> 1</a></td><td class="dataset-currentrev">2016-11-28 07:37:49</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev"></td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Lan X, Atanassov BS, Li W, Zhang Y, Florens L, Mohan RD, Galardy PJ, Washburn MP, Workman JL, Dent SYR, USP44 Is an Integral Component of N-CoR that Contributes to Gene Repression by Deubiquitinating Histone H2B. Cell Rep, 17(9):2382-2393(2016) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=27880911" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: H2B ubiquitination, N-CoR complex, USP44 deubiquitinase, Affinity Purification, MudPIT</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>principal_investigator</th></tr>
<tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Laurence Florens</th></tr>
<tr><td>contact affiliation</td><td>Stowers Institute for Medical Research</td></tr><tr><td>contact email</td><td>laf@stowers.org</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=http://massive.ucsd.edu/ProteoSAFe/status.jsp?task=a81695e3923146a285f4c15c50c29e87 target="_blank">MassIVE dataset URI</a></td></tr><tr><td><a href=ftp://massive.ucsd.edu/MSV000080144 target="_blank">Dataset FTP location</a></td></tr></table>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>