<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD000211</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD000211</h1>
    
	<div class="dataset-content">
	PXD000211 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Arabidopsis nanodomain-delimited ABA signaling pathway regulates the anion channnel SLAH3</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Analysis of Arabidopsis nanodomains shows that Arabidopsis nanodomain-delimited ABA signaling pathway regulates the anion channnel SLAH3. The phytohormone abscisic acid (ABA) plays a key role in the plant response to drought stress. Hence, ABA-dependent gene transcription and ion transport is regulated by a variety of protein kinases and phosphatases. However, the nature of the membrane delimited ABA signal transduction steps remains largely unknown. To gain insight into plasma membrane-bound ABA signaling, we identified sterol-dependent proteins associated with detergent resistant membranes from Arabidopsis thaliana mesophyll cells. Among those, we detected the central ABA signaling phosphatase ABI1 (abscisic-acid insensitive 1) and the calcium-dependent protein kinase 21 (CPK21). Using fluorescence microscopy, we found these proteins to localize in membrane nanodomains, as observed by colocalization with the nanodomain marker remorin AtRem 1.3. After transient coexpression, CPK21 interacted with SLAH3 (SLAC1 homolog 3) and activated this anion channel. Upon CPK21 stimulation, SLAH3 exhibited the hallmark properties of S-type anion channels. Coexpression of SLAH3/CPK21 with ABI1, however, prevented proper nanodomain localization of the SLAH3/CPK21 protein complex, and as a result anion channel activation failed. FRET studies revealed enhanced interaction of SLAH3 and CPK21 within the plasma membrane in response to ABA and thus confirmed our initial observations. Interestingly, the ABA-induced SLAH3/CPK21 interaction was modulated by ABI1 and the ABA receptor RCAR1/PYL9. We therefore propose that ABA signaling via inhibition of ABI1 modulates the apparent association of a signaling and transport complex within membrane domains that is necessary for phosphorylation and activation of the S-type anion channel SLAH3 by CPK21. Data processing and bioinformatics: MS result peak files were run on a Mascot daemon using the Mascot algorithm (version 2.2; Matrix Science) with TAIR v9 protein database, trypsin/P as protease. Allowed fixed modification was carbamidomethylation (C), and variable modifications were oxidized methionines (N) and pyroglutamic acid (pyro-Glu at N-terminal Q). Peptide and fragment mass tolerance were set to +-1.5 Da for the ion trap and +-0.2 Da for the Quad-TOF, maximum missed cleavages to two, and only singly, doubly, and triply charged ions were analyzed. After manual inspection, protein identifications with at least three unique identified peptides were automatically approved. Protein identifications based on two unique identified were only regarded if the two unique identified peptides displayed an ions score above the Mascot significance threshold for P < 0.05 (Mascot score >40 for the ion trap and >32 for the QUAD-TOF). Detailed information for the identified proteins was queried from the Uniprot consortium database (www.uniprot.org) and The Arabidopsis Information Center (TAIR: www.arabidopsis.org). The functional classification was assigned manually according to the annotations found in these two databases. Two independent biological replicates were analyzed: first setup of DRM isolation was performed with the detergents TritonX-100 and Brij-98 in-solution and in-gel digested. The second setup for DRM isolation was only performed with the detergent Triton X-100 and in-gel digested but subjected to methylcyclodextrin treatment to identify sterol-dependent lipid nanodomain proteins. For the proteomic analysis of DRMs data sets from both experimental setups were analyzed; the MCD treatment was only analyzed for the second setup. Protein amounts were estimated only for the MCD treatment using the Exponentially Modified Protein Abundance Index, which correlates with the number of observed and observable peptides per protein.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2013-05-02</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD000211-2&outputMode=XML&test=no' target="_blank">Submission_2013-05-02_06:37:05.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="http://dx.doi.org/10.6019/PXD000211" target="_blank">http://dx.doi.org/10.6019/PXD000211</a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Supported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Yvonne Reinders</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Arabidopsis thaliana (Mouse-ear cress);  NCBI TaxID: 3702; </td></tr><tr><td><b>ModificationList</b></td><td>monohydroxylated residue: 15.994915; iodoacetamide -site C: 57.021464; 2-pyrrolidone-5-carboxylic acid (Gln): -17.026549; iodoacetamide derivatized residue; monohydroxylated residue; neutral loss: 63.998285; deaminated residue</td></tr><tr><td><b>Instrument</b></td><td>Ion Trap; quadrupole ion trap</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2013-04-11 03:18:54</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000211-1&test=no">1</a></td><td >2013-05-02 05:34:56</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000211-1&test=no">1</a></td><td >2014-07-28 00:33:56</td><td >announced</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD000211-2&test=no"><span class='current'>&#9205;</span> 2</a></td><td class="dataset-currentrev">2013-05-02 06:37:05</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Add reference</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Demir F, Horntrich C, Blachutzik JO, Scherzer S, Reinders Y, Kierszniowska S, Schulze WX, Harms GS, Hedrich R, Geiger D, Kreuzer I, Arabidopsis nanodomain-delimited ABA signaling pathway regulates the anion channel SLAH3. Proc Natl Acad Sci U S A, 110(20):8296-301(2013) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=23630285" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: plant, arabidopsis, lipid rafts</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Yvonne Reinders</th></tr>
<tr><td>contact affiliation</td><td>University of Regensburg</td></tr><tr><td>contact email</td><td>yr@proteomics-online.de</td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/2013/05/PXD000211 target="_blank">Dataset FTP location</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/simpleSearch.do?simpleSearchValue=29252" target="_blank">29252</a><ol><li>InstrumentRef: INSTRUMENT_1</li><li>Label: EAB98ingel</li><li>ModificationList: monohydroxylated residue, iodoacetamide -site C, 2-pyrrolidone-5-carboxylic acid (Gln), </li><li>Name: Analysis of arabidopsis LR Brij98 in gel</li><li>PublicationRef: PMID23630285</li></ol></li><li><a href="http://www.ebi.ac.uk/pride/simpleSearch.do?simpleSearchValue=29256" target="_blank">29256</a><ol><li>InstrumentRef: INSTRUMENT_1</li><li>Label: EA_CMF_PM_FV</li><li>ModificationList: monohydroxylated residue, 2-pyrrolidone-5-carboxylic acid (Gln), iodoacetamide -site C, </li><li>Name: Analysis of arabidopsis LR CMF_PM_FV</li><li>PublicationRef: PMID23630285</li></ol></li><li><a href="http://www.ebi.ac.uk/pride/simpleSearch.do?simpleSearchValue=29254" target="_blank">29254</a><ol><li>InstrumentRef: INSTRUMENT_1</li><li>Label: EAinsolB98</li><li>ModificationList: monohydroxylated residue, iodoacetamide -site C, 2-pyrrolidone-5-carboxylic acid (Gln), </li><li>Name: Analysis in Arabidopsis LR in solution B98</li><li>PublicationRef: PMID23630285</li></ol></li><li><a href="http://www.ebi.ac.uk/pride/simpleSearch.do?simpleSearchValue=29259" target="_blank">29259</a><ol><li>InstrumentRef: INSTRUMENT_2</li><li>Label: EBminusMCD</li><li>ModificationList: iodoacetamide derivatized residue, monohydroxylated residue, neutral loss, deaminated residue, </li><li>Name: Exp. B -MCD</li><li>PublicationRef: PMID23630285</li></ol></li><li><a href="http://www.ebi.ac.uk/pride/simpleSearch.do?simpleSearchValue=29253" target="_blank">29253</a><ol><li>InstrumentRef: INSTRUMENT_1</li><li>Label: EATX100ingel</li><li>ModificationList: 2-pyrrolidone-5-carboxylic acid (Gln), monohydroxylated residue, iodoacetamide -site C, </li><li>Name: Analysis of arabidopsis LR TritonX100 in gel</li><li>PublicationRef: PMID23630285</li></ol></li><li><a href="http://www.ebi.ac.uk/pride/simpleSearch.do?simpleSearchValue=29255" target="_blank">29255</a><ol><li>InstrumentRef: INSTRUMENT_1</li><li>Label: EAinsolTX100</li><li>ModificationList: monohydroxylated residue, iodoacetamide -site C, 2-pyrrolidone-5-carboxylic acid (Gln), </li><li>Name: Analysis of Arabidopsis LR in sol TX100</li><li>PublicationRef: PMID23630285</li></ol></li><li><a href="http://www.ebi.ac.uk/pride/simpleSearch.do?simpleSearchValue=29258" target="_blank">29258</a><ol><li>InstrumentRef: INSTRUMENT_2</li><li>Label: EAplusMCD</li><li>ModificationList: iodoacetamide derivatized residue, monohydroxylated residue, neutral loss, deaminated residue, </li><li>Name: Exp. B + MCD</li><li>PublicationRef: PMID23630285</li></ol></li><li><a href="http://www.ebi.ac.uk/pride/simpleSearch.do?simpleSearchValue=29260" target="_blank">29260</a><ol><li>InstrumentRef: INSTRUMENT_2</li><li>Label: EBPM</li><li>ModificationList: iodoacetamide derivatized residue, monohydroxylated residue, neutral loss, deaminated residue, </li><li>Name: Exp. B PM</li><li>PublicationRef: PMID23630285</li></ol></li><li><a href="http://www.ebi.ac.uk/pride/simpleSearch.do?simpleSearchValue=29257" target="_blank">29257</a><ol><li>InstrumentRef: INSTRUMENT_1</li><li>Label: EA_Wang_PM_FV</li><li>ModificationList: monohydroxylated residue, </li><li>Name: Analysis of arabidopsis LR Wang_PM_FV</li><li>PublicationRef: PMID23630285</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>