<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD000094</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD000094</h1>
    
	<div class="dataset-content">
	PXD000094 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Proteogenomics of Tistlia consotensis</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Tistlia consotensis is a halotolerant Rhodospirillaceae that was isolated from a saline spring located in the Colombian Andes with a salt concentration close to seawater (4.5%w/vol). We cultivated this microorganism in three NaCl concentrations, i.e. optimal (0.5%), without (0.0%) and high (4.0%) salt concentration, and analyzed its cellular proteome. For assigning tandem mass spectrometry data, we first sequenced its genome and constructed a six reading frame ORF database from the draft sequence. We annotated only the genes whose products (872) were detected. We compared the quantitative proteome data sets recorded for the three different growth conditions.Peak lists were generated with the MASCOT DAEMON software (version 2.3.2) from Matrix Science using the extract_msn.exe data import filter from the Xcalibur FT package (version 2.0.7) proposed by ThermoFisher. Data import filter options were set at 400 (minimum mass), 5,000 (maximum mass), 0 (grouping tolerance), 0 (intermediate scans), and 1,000 (threshold). MS/MS spectra were searched against the  home-made ORF database with the following parameters: tryptic peptides with a maximum of 2 miss cleavages during proteolytic digestion, a mass tolerance of 5 ppm on the parent ion and 0.5 Da on the MS/MS, fixed modification for carbamidomethylated Cys (+57.0215) and variable modification for oxidized Met (+15.9949).  All peptide matches with a peptide score above its query threshold set at p < 0.05 with the ORF database and rank 1 were parsed using the IRMa 1.28.0 software. False-positive rate for peptide identification was estimated using a decoy database as below 0.5% with these parameters. MS/MS spectra assigned to several loci were systematically removed. A protein was considered validated when at least two different peptides were detected in the same experiment. False-positive identification of proteins was estimated using a reverse decoy database as below 0.1% with these parameters The number of MS/MS spectra per protein (spectral counts) was determined for the three replicates in each growth condition. The protein abundances were compared using the T-Fold option of the PatternLab 2.0 software. This module allows normalising the spectral count datasets, calculating the average fold changes with statistics (t-test), and estimating the resulting theoretical false discovery rate. Stringent parameters were used in this analysis: minimum fold change of 1.5, minimum p-value of 0.05 and BH-FDR Alfa of 0.15.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2014-07-28</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD000094-2&outputMode=XML&test=no' target="_blank">Submission_2014-07-28_00:26:34.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="http://dx.doi.org/10.6019/PXD000094" target="_blank">http://dx.doi.org/10.6019/PXD000094</a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Supported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Jean ARMENGAUD</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Bacteria;  NCBI TaxID: 2; </td></tr><tr><td><b>ModificationList</b></td><td>monohydroxylated residue; iodoacetamide derivatized residue</td></tr><tr><td><b>Instrument</b></td><td>LTQ Orbitrap; instrument model: LTQ Orbitrap</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2012-12-11 03:01:48</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000094-1&test=no">1</a></td><td >2012-12-18 03:05:23</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000094-1&test=no">1</a></td><td >2014-05-21 06:33:37</td><td >announced</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD000094-2&test=no"><span class='current'>&#9205;</span> 2</a></td><td class="dataset-currentrev">2014-07-28 00:26:35</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Updated project metadata.</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Rubiano-Labrador C, Bland C, Miotello G, Guérin P, Pible O, Baena S, Armengaud J, Proteogenomic insights into salt tolerance by a halotolerant alpha-proteobacterium isolated from an Andean saline spring. J Proteomics, 97():36-47(2014) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=23727365" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: Proteogenomics, Draft genome, Halotolerance, Alphaproteobacteria, Rhodospirillaceae</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Jean ARMENGAUD</th></tr>
<tr><td>contact affiliation</td><td>CEA-DSV-iBEB-SBTN-LBSP</td></tr><tr><td>contact email</td><td>jean.armengaud@cea.fr</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2012/12/PXD000094 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD000094 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD000094" target="_blank">PXD000094</a><ol><li>Label: PRIDE project</li><li>Name: Proteogenomics of Tistlia consotensis</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>