<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD000316</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD000316</h1>
    
	<div class="dataset-content">
	PXD000316 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>A Saccharomyces Cerevisiae model reveals in vivo functional impairment of the ogden syndrome n-terminal acetyltransferase Naa10-S37p mutant</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Abstract: N-terminal acetylation (Nt-acetylation) occurs on the majority of eukaryotic proteins and is catalysed by N-terminal acetyltransferases (NATs). Nt- acetylation is increasingly recognized as a vital modification with functional implications ranging from protein degradation to protein localization.  Very recently, the first human X-linked genetic disorder caused by a mutation in a NAT gene was reported; Ogden syndrome boys harbour a p.Ser37Pro  variant in the gene encoding Naa10, the catalytic subunit of the NatA complex, and suffer from global developmental delays and lethality during infancy.  Here, we developed a Saccharomyces cerevisiae model by introducing the human wildtype or mutant NatA complex into yeast lacking NatA (NatA-∆). The human  NatA complex phenotypically complemented the NatA-∆ strain, while only a partial rescue was observed for the Ogden mutant NatA complex suggesting that  hNaa10-S37P is only partially functional in vivo. Furthermore, we performed quantitative Nt-acetylome analyses on a control yeast strain (yNatA), a yeast  NatA deletion strain (yNatA-∆), a yeast NatA deletion strain expressing human NatA (hNatA), and a yeast NatA deletion strain expressing mutant human NatA  (hNatA-hNaa10-S37P). Interestingly, a reduced degree of Nt-acetylation was specifically observed among a large group of NatA substrates in the yeast  expressing mutant hNatA as compared to yeast expressing wildtype hNatA. Furthermore, immunoprecipitated mutant NatA complex displayed a reduced catalytic  activity in vitro as compared to the wildtype NatA complex. Combined, these data provide strong support for the functional impairment of hNaa10-S37P in  vivo and suggest that reduced Nt-acetylation of one or more target substrates contributes to the pathogenesis of Ogden syndrome. Comparative analysis  between human and yeast NatA also provided novel insights into the co-evolution of the NatA complexes and their substrates. For instance, (Met-)Ala- N- termini are more prevalent in the human proteome as compared to the yeast proteome, and hNatA displays a relative preference towards these N-termini as  compared to yNatA.    Methods: The obtained peptide mixtures were introduced into an LC-MS/MS system, the Ultimate 3000 (Dionex, Amsterdam, The Netherlands) in-line connected to an LTQ  Orbitrap XL (Thermo Fisher Scientific, Bremen, Germany). Samples were first loaded on a trapping column (made in-house, 100 µm internal diameter (I.D.) x  20 mm, 5 µm beads C18 Reprosil-HD, Dr. Maisch). After back-flushing from the trapping column, the sample was loaded on a reverse-phase column (made in- house, 75 µm I.D. x 150 mm , 5 µm beads C18 Reprosil-HD, Dr. Maisch). Peptides were loaded with solvent A (0.1% trifluoroacetic acid, 2% acetonitrile),  and were separated with a linear gradient from 2% solvent A’ (0.05% formic acid) to 55% solvent B’ (0.05% formic acid and 80% acetonitrile) at a flow  rate of 300 nl/min followed by a wash reaching 100% solvent B’. The mass spectrometer was operated in data-dependent mode, automatically switching  between MS and MS/MS acquisition for the six most abundant peaks in a given MS spectrum. Full scan MS spectra were acquired in the Orbitrap at a target  value of 1E6 with a resolution of 60,000. The six most intense ions were then isolated for fragmentation in the linear ion trap, with a dynamic exclusion  of 60 s. Peptides were fragmented after filling the ion trap at a target value of 1E4 ion counts. From the MS/MS data in each LC run, Mascot Generic  Files were created using the Mascot Distiller software (version 2.3.01, Matrix Science). While generating these peak lists, grouping of spectra was  allowed with a maximum intermediate retention time of 30 s and a maximum intermediate scan count of 5 was used where possible. Grouping was done with  0.005 Da precursor tolerance. A peak list was only generated when the MS/MS spectrum contained more than 10 peaks. There was no de-isotoping and the  relative signal to noise limit was set at 2. These peak lists were then searched with the Mascot search engine (Matrix Science) using the Mascot Daemon  interface (version 2.3, Matrix Science).  Spectra were searched against the yeast (S. cerevisiae) Swiss-Prot database (version 2011_03  of the UniProtKB/Swiss-Prot protein database containing  7,320 yeast sequence entries (525997 sequences in total)). )). 13C2D3- [(13C2)-trideutero-acetylation] at lysines, carbamidomethylation of cysteine and  methionine oxidation to methionine-sulfoxide were set as fixed modifications for the N-terminal COFRADIC analyses. Variable modifications were 13C2D3-  acetylation and acetylation of protein N-termini. Pyroglutamate formation of N-terminal glutamine was additionally set as a variable modification. Mass  tolerance on precursor ions was set to 10 ppm (with Mascot’s C13 option set to 1) and on fragment ions to 0.5 Da. Endoproteinase semi-Arg-C/P (Arg-C  specificity with arginine-proline cleavage allowed) was set as enzyme allowing no missed cleavages. The peptide charge was set to 1+, 2+, 3+ and  instrument setting was put on ESI-TRAP. Only peptides that were ranked one and scored above the threshold score, set at 99% confidence, were withheld.  The estimated false discovery rate by searching decoy databases was typically found to lie between 2 and 4% on the spectrum level [33]. Quantification of  the degree of Nt-Ac was performed as described previously (5). All data management was done in ms_lims ([49]).    Please note!! the pride result files will show a lot of delta m/z deviations. The modification for the heavy acetylation (+47 Da) used in this experiment is not in Pride. So another heavy acetylation (+45 Da) modification was annotated when creating the xml files.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2014-07-28</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD000316-2&outputMode=XML&test=no' target="_blank">Submission_2014-07-28_00:40:59.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="http://dx.doi.org/10.6019/PXD000316" target="_blank">http://dx.doi.org/10.6019/PXD000316</a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Supported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Pieter-Jan De Bock</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Saccharomyces cerevisiae (Baker's yeast);  NCBI TaxID: 4932; </td></tr><tr><td><b>ModificationList</b></td><td>L-methionine sulfoxide: 15.994915; 2-pyrrolidone-5-carboxylic acid (Gln): -17.026549; iodoacetamide derivatized residue: 57.021464; N-acetylated residue: 42.010565; acetate labeling reagent (N-term) (heavy form: +3amu)</td></tr><tr><td><b>Instrument</b></td><td>LTQ Orbitrap; instrument model: LTQ-Orbitrap</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2013-07-02 07:33:38</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000316-1&test=no">1</a></td><td >2014-01-29 06:09:09</td><td >announced</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD000316-2&test=no"><span class='current'>&#9205;</span> 2</a></td><td class="dataset-currentrev">2014-07-28 00:41:00</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Updated project metadata.</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Van Damme P, Støve SI, Glomnes N, Gevaert K, Arnesen T, A Saccharomyces cerevisiae model reveals in vivo functional impairment of the Ogden syndrome N-terminal acetyltransferase NAA10 Ser37Pro mutant. Mol Cell Proteomics, 13(8):2031-41(2014) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=24408909" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: Yeast, LC-MSMS, N-terminal COFRADIC</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Pieter-Jan De Bock</th></tr>
<tr><td>contact affiliation</td><td>VIB UGent Department of medical protein research</td></tr><tr><td>contact email</td><td>pieterj.debock@ugent.be</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2014/01/PXD000316 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD000316 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD000316" target="_blank">PXD000316</a><ol><li>Label: PRIDE project</li><li>Name: A Saccharomyces Cerevisiae model reveals in vivo functional impairment of the ogden syndrome n-terminal acetyltransferase Naa10-S37p mutant</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>