<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD008711</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD008711</h1>
    
	<div class="dataset-content">
	PXD008711 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Searching NE-enriched proteomics datasets for splice junction peptides</td></tr><tr><td><b>Description</b></td><td class='breakwords'>MudPIT datasets for NaOH-extracted and Salt-and-Detergent (SD)-extracted rat nuclear membranes were obtained from skeletal muscle and liver tissue as described in the following references:
Wilkie GS, Korfali N, Swanson SK, Malik P, Srsen V, Batrakou DG, de las Heras J, Zuleger N, Kerr AR, Florens L, Schirmer EC. Several novel nuclear envelope transmembrane proteins identified in skeletal muscle have cytoskeletal associations Mol Cell Proteomics 2011 Jan;10(1):M110003129
Korfali N, Wilkie GS, Swanson SK, Srsen V, de Las Heras J, Batrakou DG, Malik P, Zuleger N, Kerr AR, Florens L, Schirmer EC. The nuclear envelope proteome differs notably between tissues Nucleus 2012 Nov-Dec;3(6):552-64

The trypsin-digested MS datasets from these studies were searched again using ProLuCID against a database containing tissue-specific junction sequences as such:
We analyzed RNA-Seq data from five rat tissues (heart, skeletal muscle, liver, brain, and testes) produced by two separate research groups, hereby referred to as GSE4 (Yu, Y., J.C. Fuscoe, C. Zhao, C. Guo, M. Jia, T. Qing, et al., A rat RNA-Seq transcriptomic BodyMap across 11 organs and 4 developmental stages. Nat Commun, 2014; 5: 3230.) and GSE5 (Merkin, J., C. Russell, P. ChenC.B. Burge, Evolutionary dynamics of gene and isoform regulation in Mammalian tissues. Science, 2012; 338(6114): 1593-9.)

GSE4 has higher read coverage over junctions, but only one suitable sample per tissue while GSE5 has lower coverage but many replicates. Novel junctions identified in both datasets were considered to be high confidence. Differentially spliced transcripts were determined and quantified using Modeling Alternative Junction Inclusion Quantification (MAJIQ) software. Splice junctions rather than whole isoforms were quantified due to the known limitations in reconstructing whole transcripts from short read data. The software builds splice graphs from RNA-Seq reads spanning spliced junctions, incorporates un-annotated junctions, and uses information from replicate samples to build a Bayesian posterior distribution, outputting the expected percentage spliced in values (E|PSI|) corresponding to the percentage of transcripts that are expected to contain the given splice junction or in the case of a comparison, change in PSI (E|dPSI). In accordance with previous literature, we considered an alternative splicing event between tissues when E|dPSI greater than 0.2.

Tissue-specific junction sequence databases were generated for rat muscle and liver tissues. In each case the union of all splice junctions detected by STAR aligner for each replicate from GSE4 and GSE5 was taken. Junctions were filtered to remove junctions supported by less than 6 reads and junctions predicting an intron length of less than 60 nucleotides (a standard cut-off as there are very few smaller annotated junctions). Junction coordinates were extended by 66 nucleotides in both directions, and then translated in three frames according to the directionality of the gene. This produced peptide sequences of about 44 amino acids in length. Sequences were removed if the translation produced a stop codon before the junction based on standard practice. Novel exons and intron retentions predicted by MAJIQ were similarly translated in three frames and added to the database, but were removed if the translated sequence was less than 7 amino acids long. All novel and annotated junctions were combined with novel exons, intron retentions, and Ensembl rat protein database sequences to produce a final protein sequence database. Finally, coordinates of junction peptides were checked against the original fasta sequence to be sure that the peptide crossed the junction with a start less or equal to 22 amino acids away and an end greter or equal to 22 amino acids away.</td></tr><tr><td><b>HostingRepository</b></td><td>MassIVE</td></tr><tr><td><b>AnnounceDate</b></td><td>2018-04-20</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD008711-1&outputMode=XML&test=no' target="_blank">Submission_2018-04-20_15:03:08.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Non peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Laurence Florens</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Rattus norvegicus;  common name: Norway rat;  NCBI TaxID: 10116; </td></tr><tr><td><b>ModificationList</b></td><td>S-carboxamidomethyl-L-cysteine; L-methionine sulfoxide</td></tr><tr><td><b>Instrument</b></td><td>LTQ</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2018-01-18 09:32:25</td><td >ID requested</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD008711-1&test=no"><span class='current'>&#9205;</span> 1</a></td><td class="dataset-currentrev">2018-04-20 15:03:09</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev"></td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>no publication</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: nuclear envelope, nuclear envelope transmembrane protein (NET), tissue-specific, splice variant, nuclear envelopathies, muscular dystrophy</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Laurence Florens</th></tr>
<tr><td>contact affiliation</td><td>The Stowers Institute for Medical Research</td></tr><tr><td>contact email</td><td>laf@stowers.org</td></tr><tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Laurence Florens</th></tr>
<tr><td>contact affiliation</td><td>Stowers Institute for Medical Research</td></tr><tr><td>contact email</td><td>laf@stowers.org</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=http://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=a51682db240d4bc4b93fb9ff65405a9d target="_blank">MassIVE dataset URI</a></td></tr><tr><td><a href=ftp://massive.ucsd.edu/MSV000081951 target="_blank">Dataset FTP location</a></td></tr></table>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>