<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD008153</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD008153</h1>
    
	<div class="dataset-content">
	PXD008153 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Integration of multilevel OMICs data based on the identification of regulatory modules from protein-protein interaction networks</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Complex scientific experiments provide researchers with a wealth of data and knowledge from heterogeneous sources. Analyzed in its entirety, OMICs data provide a deep insight into the overall biological processes of organisms. However, the integration of data from different cellular levels (e.g., transcriptomics and proteomics) is challenging. Analyzing lists of differentially abundant molecules from different cellular levels often results in a small overlap, which can be accounted to, e.g., different regulatory mechanisms, different temporal scales as well as inherent properties of the measurement method. Thus, there is a need for approaches that allow efficient integration of OMICs data from different cellular levels. In this study, we make use of transcriptome, proteome and secretome data from the human pathogenic fungus Aspergillus fumigatus challenged with the antifungal drug caspofungin. Caspofungin targets the fungal cell wall leading to a compensatory stress response. We analyze the experimental data based on two different approaches. First, we apply a simple approach based on the comparison of differentially regulated genes and proteins with subsequent pathway analysis. Second, we compare the cellular levels based on the identification of regulatory or functional modules by two module-detecting algorithms from protein-protein interaction networks in conjunction with transcriptomic and proteomic data. Our results show that both approaches associate the fungal caspofungin response with biological pathways like cell wall biosynthesis, fatty acid metabolism as well as carbohydrate metabolism. Compared to results of the simple approach, the use of regulatory modules shows a notably higher agreement between the different cellular levels. The additional structural information of the networks provided by the module-based approach allows for topological analysis as well as the analysis of the temporal evolution of cellular response at a molecular level. However, we also found that quality of the module-based results depends on the comprehensiveness of the underlying protein-protein interaction network itself. Thus, while our results highlight the benefits and potential provided by a module-based analysis of OMICs data from different cellular levels, future studies will have to focus on the expansion of organism specific protein-protein interaction networks.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2018-10-25</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD008153-1&outputMode=XML&test=no' target="_blank">Submission_2018-10-25_05:45:27.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Thomas Krüger</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Neosartorya fumigata (Aspergillus fumigatus);  NCBI TaxID: 746128; </td></tr><tr><td><b>ModificationList</b></td><td>TMT6plex-126 reporter+balance reagent acylated residue; iTRAQ4plex-116 reporter+balance reagent acylated residue; monohydroxylated residue; iodoacetamide derivatized residue</td></tr><tr><td><b>Instrument</b></td><td>Q Exactive</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2017-11-07 04:46:16</td><td >ID requested</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD008153-1&test=no"><span class='current'>&#9205;</span> 1</a></td><td class="dataset-currentrev">2018-10-25 05:45:29</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev"></td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Conrad T, Kniemeyer O, Henkel SG, Krüger T, Mattern DJ, Valiante V, Guthke R, Jacobsen ID, Brakhage AA, Vlaic S, Linde J, Module-detection approaches for the integration of multilevel omics data highlight the comprehensive response of Aspergillus fumigatus to caspofungin. BMC Syst Biol, 12(1):88(2018) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=30342519" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>curator keyword: Technical</td></tr><tr><td>submitter keyword: Aspergillus fumigatus, proteomics, secretome, LC-MS, caspofungin</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Olaf Kniemeyer</th></tr>
<tr><td>contact affiliation</td><td>Leibniz Institute for Natural Product Research and Infection Biology, Department of Molecular and Applied Microbiology, Stress- and Immunoproteomics group</td></tr><tr><td>contact email</td><td>olaf.kniemeyer@leibniz-hki.de</td></tr><tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Thomas Krüger</th></tr>
<tr><td>contact affiliation</td><td>Leibniz Institute for Natural Product Research and Infection Biology - Hans Knöll Institute</td></tr><tr><td>contact email</td><td>thomas.krueger@leibniz-hki.de</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2018/10/PXD008153 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD008153 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD008153" target="_blank">PXD008153</a><ol><li>Label: PRIDE project</li><li>Name: Integration of multilevel OMICs data based on the identification of regulatory modules from protein-protein interaction networks</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>