<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD000273</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD000273</h1>
    
	<div class="dataset-content">
	PXD000273 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Quantitative proteomics reveals the role of arginine phosphorylation in bacterial stress response</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Application of a novel TiO2-based phosphopeptide enrichment protocol to study the role of protein arginine phosphorylation in B.subtilis.  1 iTRAQ Quantification of pArg peptides in B.subtilis upon heat shock and oxidative stress in comparison to unstressed control 2 Studying pArg upon deletion of McsB protein arginine kinase in B.subtilis (double mutant cells for McsB and YwlE phosphoarginine phosphatase) 3 Studying pArg in wt cells upon inhibition of YwlE phosphatase with cell wall permeable inihbitor sodium pervanadate  Short overview McsB is a bacterial protein arginine kinase that is involve in stress adaptation for heat shock and oxidative stress  - cells were grown in LB medium until O.D. 0.6 and split into 3 fractions  - one fraction was treated with heat stress for 20 min at 50 degrees C, one was stressed with 1mM diamide (induced oxidative foldeing stress), and one was kept at 37 degrees C (control) - cell harvest, lysis with a modified FASP protocol including tryptic cleavage - iTRAQ labelling for some experiments - TiO2 enrichment to purify all phosphopeptides - LC-MS/MS analysis of phosphopetides - phosphosite localization by phosphoRS and manually - quantitation of phosphorylation relative to protein abundance in the different samples. Peptide and protein identification: Raw data were extracted by the Protein Discoverer software suite (version 1.2.227 for wt phosphorylation, all 4 replicates of the iTRAQ/TiO2 pArg study and replicates 1 and 2 for the pervanadate and the double mutant experiments and version 1.3.0.339 for protein abundance determination, and replicates 3 and 4 of the pervanadate and double mutant experiments. Thermo-Fisher Scientific) and searched against a combined forward/reversed database of B. subtilis protein sequences and common contaminants using MASCOT (version 2.2, www.matrixscience.com). Since cells were grown in LB medium, all raw data were independently searched against a combined B. subtilis and yeast (S. cerevisae) database which did not result in a significant increase in protein hits or identification of yeast proteins. Carbamidomethylation of cysteine, iTRAQ modification of peptide Nterminus and lysine;-amino group were set as fixed modifications. Phosphorylation of serine, threonine, tyrosine, histidine, and arginine as well as methionine sulfoxidation were selected to be variable, with a maximum of 4 modifications per peptide. Since tryptic cleavage is impaired at phosphorylated arginine, a maximum of two missed cleavage sites was allowed, whereas fully tryptic cleavage of both termini was required. The peptide mass deviation was set to 4 ppm; fragment ions were allowed to have a mass deviation of 0.5 Da for CID and ETD data and 0.025 Da for HCD data. False discovery rates were assessed from reversed database hits including all identified peptide spectrum matches with MASCOT score above 19, rank 1, and a peptide length of at least 6 amino acids. This resulted in false discovery rates below 1 % on the peptide spectrum match level and below 1.5% on the unique peptide level for each experiment. For reliable phosphorylation site analysis, all phosphopeptide hits were automatically re-analyzed by the phosphoRS software (Taus et al, 2011) within the Protein Discoverer software suite, manually validated, and compared to known B. subtilis protein phosphorylations (Macek et al, 2007). Proteins were grouped according to their biological function assignment on the SubtiList homepage (http://genolist.pasteur.fr/SubtiList/). Relative quantification of phosphorylation: Peptide quantification was achieved by the reporter ions quantifier node in Protein Discoverer. This node extracted reporter ion areas from corresponding HCD scans for all identified peptide hits (HCD, CID and ETD) based on the precursor mass. Extracted peak areas for heat shock and oxidative stress were divided by the peak area of the control sample to give the corresponding peptide abundance ratio. Regulatory cut-offs for peptide abundance ratios were determined from the distribution of unphosphorylated peptides. A 95% confidence window was applied to log2 normalized values of peptide regulations and all hits outside of this window were considered to be regulated. For the heat stress/control ratio regulatory cut-off values of 0.59 and 1.75 were determined. Peptides with abundance changes within 0.30 to 2.50 for the oxidative stress/control ratio were not considered to be regulated. Protein regulations were the averages of all identified and quantified unphosphorylated peptides from all analyzed samples (Supplementary Table 5). Phosphopeptide abundances were calculated by averaging, if a phosphopeptide was identified in several replicates and/or charge states. To argument for upregulation of phosphorylation, each phospho-peptide ratio was divided by the respective protein abundance ratio, in case the protein was identified before. For regulation of the phosphorylation, we oriented the regulatory cut-offs determined before. We compared each phosphopeptide abundance against the respective protein abundance and determined a twofold regulation to be significant.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2014-07-24</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD000273-3&outputMode=XML&test=no' target="_blank">Submission_2014-07-24_03:47:20.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Andreas Schmidt</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Bacillus subtilis subsp. subtilis str. 168;  NCBI TaxID: 224308; </td></tr><tr><td><b>ModificationList</b></td><td>monohydroxylated residue; iodoacetamide derivatized residue; phosphorylated residue; iTRAQ4plex-116 reporter+balance reagent acylated residue</td></tr><tr><td><b>Instrument</b></td><td>LTQ Orbitrap Velos; Q Exactive</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2013-05-31 01:29:29</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000273-1&test=no">1</a></td><td >2013-11-21 02:46:09</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000273-2&test=no">2</a></td><td >2014-05-21 03:14:15</td><td >announced</td><td >Updated publication reference for PubMed record(s): 24263382.</td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD000273-3&test=no"><span class='current'>&#9205;</span> 3</a></td><td class="dataset-currentrev">2014-07-24 03:47:21</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Updated project metadata.</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Schmidt A, Trentini DB, Spiess S, Fuhrmann J, Ammerer G, Mechtler K, Clausen T, Quantitative phosphoproteomics reveals the role of protein arginine phosphorylation in the bacterial stress response. Mol Cell Proteomics, 13(2):537-50(2014) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=24263382" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>curator keyword: Biological</td></tr><tr><td>submitter keyword: Arginine phosphorylation, stress response, Bacillus subtilis</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Andreas Schmidt</th></tr>
<tr><td>contact affiliation</td><td>Adolf-Butenandt-Institut</td></tr><tr><td>contact email</td><td>A.Schmidt@med.uni-muenchen.de</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2013/11/PXD000273 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD000273 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD000273" target="_blank">PXD000273</a><ol><li>Label: PRIDE project</li><li>Name: Quantitative proteomics reveals the role of arginine phosphorylation in bacterial stress response</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>