<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD026571</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD026571</h1>
    
	<div class="dataset-content">
	PXD026571 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>MudPIT analyses of the proteins associated with Human SETD2 deletion mutants</td></tr><tr><td><b>Description</b></td><td class='breakwords'>N-terminally Halo-tagged deletions of human SETD2 were constructed such as SETD2 ABCN1, A, B, C, CN, and CC consisted of amino acids 1-1692, 1-503, 504-1403, 1404-2564, 1404-1963, and 1964-2564, respectively. Deletion mutants were expressed in HEK293T cells and affinity purified, each in biological duplicates, along with two negative controls. Eluted proteins were TCA precipitated and analyzed independently by Multidimensional Protein Identification Technology (MudPIT). Briefly, precipitated protein samples were resuspended in 100mM Tris pH 8.5, 8M urea to denature the proteins.  Proteins were reduced and alkylated prior to digestion with recombinant LysC (Promega) and trypsin (Promega). Reactions were quenched by the addition of formic acid to a final concentration of 5%.  Peptide samples were pressure-loaded onto 100 um fused silica microcapillary columns packed first with 9 cm of Aqua C18 reverse phase material, followed by 3 cm of 5-um Luna strong cation exchange material followed by 1 cm of 5-um C18 RP. The loaded microcapillary columns were placed in-line with 1200 or 1260 quaternary HPLCs. The application of a 2.5 kV distal voltage electrosprayed the eluting peptides directly into LTQ linear ion trap or Velos-Orbitraps mass spectrometers equipped with a custom-made nano-LC electrospray ionization source. Full MS spectra were recorded on the eluting peptides over a 400 to 1600 m/z range, followed by fragmentation in the ion trap (at 35% collision energy) on the first to fifth most intense ions selected from the full MS spectrum. Dynamic exclusion was enabled for 120 sec. Mass spectrometer scan functions and HPLC solvent gradients were controlled by the XCalibur data system.
RAW files were extracted into .ms2 file format using RawDistiller v. 1.0, in-house developed software. MS/MS spectra were searched using ProLuCID with a 500 ppm mass tolerance for peptide and fragment ions. Trypsin specificity was imposed on both ends of candidate peptides during the search against a protein database combining 44,080 human proteins (NCBI 2019-11-03 release), the amino acid sequences for the SETD2 deletion mutants, as well as 426 common contaminants such as human keratins, IgGs, and proteolytic enzymes. To estimate false discovery rates (FDR), each protein sequence was randomized (keeping the same amino acid composition and length) and the resulting "shuffled" sequences were added to the database, for a total search space of 89,038 amino acid sequences. A mass of 57.0125 Da was added as a static modification to cysteine residues and 15.9949 Da was differentially added to methionine residues.
DTASelect v.1.9 was used to select and sort peptide/spectrum matches (PSMs) passing the following criteria set: PSMs were only retained if they had a DeltCn of at least 0.08; minimum XCorr values of 2.1 for singly-, 2.7 for doubly-, and 3.2 for triply-charged spectra; peptides had to be at least 7 amino acids long. Results from each sample were merged and compared using CONTRAST. Combining all replicates, proteins had to be detected by at least 2 peptides and/or 2 spectral counts. Proteins that were subsets of others were removed using the parsimony option in DTASelect on the proteins detected after merging all runs. Proteins that were identified by the same set of peptides (including at least one peptide unique to such protein group to distinguish between isoforms) were grouped together, and one accession number was arbitrarily considered as representative of each protein group. 
</td></tr><tr><td><b>HostingRepository</b></td><td>MassIVE</td></tr><tr><td><b>AnnounceDate</b></td><td>2021-08-04</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD026571.0-1&outputMode=XML&test=no' target="_blank">Submission_2021-08-04_08:54:32.698.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Non peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Supported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Stowers Proteomics</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Homo sapiens;  common name: human;  NCBI TaxID: 9606; </td></tr><tr><td><b>ModificationList</b></td><td>S-carboxamidomethyl-L-cysteine; L-methionine sulfoxide</td></tr><tr><td><b>Instrument</b></td><td>LTQ; LTQ Orbitrap Velos</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2021-06-08 12:20:54</td><td >ID requested</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD026571-1&test=no"><span class='current'>&#9205;</span> 1</a></td><td class="dataset-currentrev">2021-08-04 08:54:33</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev"></td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>no publication</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: SETD2 methyltransferase, intrinsically disordered region (IDR), protein degradation</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Laurence Florens</th></tr>
<tr><td>contact affiliation</td><td>Stowers Institute for Medical Research</td></tr><tr><td>contact email</td><td>laf@stowers.org</td></tr><tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Stowers Proteomics</th></tr>
<tr><td>contact affiliation</td><td>Stowers Institute for Med Res</td></tr><tr><td>contact email</td><td>proteomicslab@stowers.org</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=http://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=2c7dbc5ddab843c5ae1de7536c347fad target="_blank">MassIVE dataset URI</a></td></tr><tr><td><a href=ftp://massive.ucsd.edu/MSV000087587/ target="_blank">Dataset FTP location</a></td></tr></table>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>