<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD000293</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD000293</h1>
    
	<div class="dataset-content">
	PXD000293 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Ti4+-IMAC label-free quantification</td></tr><tr><td><b>Description</b></td><td class='breakwords'>We combine high-resolution mass spectrometry with Ti4+-IMAC phosphopeptide enrichment and label-free quantification to monitor the phosphoproteome of Jurkat T-cells following stimulation by Prostaglandin E2. Jurkat T lymphoma cells were grown in RPMI 1640 medium supplemented with 10% fetal bovine serum and penicillin/streptomycin (Lonza). For PGE2 stimulation, cells were centrifuged for 1 min at 1500g. The growth medium was removed and the cells were resuspended at a final concentration of 1-2 × 106 cells/ml in RMPI and supplemented with 0 (control) or 10 uM PGE2 and incubated for 5, 10, 20, 30 or 60 min. After cell lysis, reduction and alkylation, proteins were digested with Lys-C and Trypsin. The digests were desalted using Sep-Pak C18 cartridges, dried in vacuo and stored at −80 °C for further use. Phosphopeptide enrichment was performed as described in Zhou H, Ye M, Dong J, Corradini E, Cristobal A, Heck AJ, Zou H, Mohammed S. Nat Protoc. 2013 Mar;8(3):461-80. The enriched phosphopeptides were subjected to a reversed phase nano-LC–MS/MS analysis consisting of a Proxeon EASY-nLC 1000, an analytical column heater (40°C) and an LTQ-Orbitrap Elite. After the survey scans, the 20 most intense precursors were selected for subsequent CID or ETD-IT fragmentation. A programmed data-dependent decision tree determined the choice of the most appropriate technique for a selected precursor. In essence, doubly charged peptides were subjected to CID fragmentation and more highly charged peptides were fragmented using ETD. Raw data were processed with MaxQuant version 1.3.0.523, and the peptides were identified from the MS and MS/MS spectra searched against a concatenated forward-decoy Swissprot Homo sapiens database version 2012_09 (40,992 sequences) using the Andromeda search engine. The database search was performed with the following parameters: an initial mass tolerance of ±20 ppm for precursor masses; final mass tolerance of ±6 ppm: ±0.6 Da for CID and ETD-ion trap fragment ions, allowing two missed cleavages. Cysteine carbamidomethylation was used as a fixed modification and methionine oxidation, protein N-terminal acetylation and serine, threonine and tyrosine phosphorylation as variable modifications. For the identification, the false discovery rate was set to 0.01 for peptides, proteins and sites and the minimum peptide length allowed was six amino acids and a minimum peptide score of 60. The match between run feature was set on. A site localization probability of at least 0.75 and a score difference of at least 5 were used as threshold for the phosphoresidue localization. Normalization was performed by subtracting the median of log transformed intensities for each LC-MS/MS run.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2017-10-24</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD000293-5&outputMode=XML&test=no' target="_blank">Submission_2017-10-24_00:09:57.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Piero Giansanti</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Homo sapiens (Human);  NCBI TaxID: 9606; </td></tr><tr><td><b>ModificationList</b></td><td>phosphorylated residue; acetylated residue; monohydroxylated residue; iodoacetamide derivatized residue</td></tr><tr><td><b>Instrument</b></td><td>LTQ Orbitrap Elite</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2013-06-21 01:47:02</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000293-1&test=no">1</a></td><td >2014-05-22 03:11:39</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000293-2&test=no">2</a></td><td >2014-05-27 07:51:26</td><td >announced</td><td >Updated publication reference for PubMed record(s): 24850871.</td></tr>
<tr><td ><a href="GetDataset?ID=PXD000293-3&test=no">3</a></td><td >2014-07-24 05:18:16</td><td >announced</td><td >Updated project metadata.</td></tr>
<tr><td ><a href="GetDataset?ID=PXD000293-4&test=no">4</a></td><td >2017-06-21 00:30:47</td><td >announced</td><td >Updated project metadata.</td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD000293-5&test=no"><span class='current'>&#9205;</span> 5</a></td><td class="dataset-currentrev">2017-10-24 00:09:58</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Updated project metadata.</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>de Graaf EL, Giansanti P, Altelaar AF, Heck AJ, Single-step enrichment by Ti4+-IMAC and label-free quantitation enables in-depth monitoring of phosphorylation dynamics with high reproducibility and temporal resolution. Mol Cell Proteomics, 13(9):2426-34(2014) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=24850871" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>ProteomeXchange project tag: PRIME-XS Project</td></tr><tr><td>curator keyword: Biological</td></tr><tr><td>submitter keyword: Label-free phosphoproteomics</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Albert J. R. Heck</th></tr>
<tr><td>contact affiliation</td><td>Biomolecular Mass Spectrometry and Proteomics, Bijvoet Center for Biomolecular Research and Utrecht Institute for Pharmaceutical Sciences, Utrecht University, Padualaan 8, 3584 CH Utrecht, The Netherlands</td></tr><tr><td>contact email</td><td>A.J.R.Heck@uu.nl</td></tr><tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Piero Giansanti</th></tr>
<tr><td>contact affiliation</td><td>NPC</td></tr><tr><td>contact email</td><td>p.giansanti@uu.nl</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2014/05/PXD000293 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD000293 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD000293" target="_blank">PXD000293</a><ol><li>Label: PRIDE project</li><li>Name: Ti4+-IMAC label-free quantification</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>