<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD000132</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD000132</h1>
    
	<div class="dataset-content">
	PXD000132 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Analysis of lung cancer cell lines GLC1 and GLC1 M13</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Datasets contain protein identification and raw data from MALDI protein identifications. Identified proteins resulted from a comparison of GLC1 and GLC1 SCLC cell lines using 2D DIGE. MALDI-TOF-MS analyses were performed on an UltraFlexTM II (Bruker Daltonics) instrument according to the instructions of the manufacturer. The instrument was equipped with a scoutTM MTP MALDI target. The spectra were acquired in the positive ion mode according to the settings given by the manufacturer. For external calibration, a peptide standard (m/z 757.399, 1296.684, 1619.822, 2093.086 and 3147.471) was used. The MALDI-PMF spectra were processed using the FlexAnalysis™ 2.4 software (Bruker Daltonics) and converted in the .xml format. For peak detection, the spectra were subjected to an internal recalibration using 13 different monoisotopic masses from autolysis products of trypsin and fragments of keratins ranging from m/z 842.509 – 2825.406. Following parameters were applied: snap peak detection algorithm, signal to noise threshold of 6, maximal number of peaks 100, quality factor threshold 50 and baseline subtraction TopHat. The generated mass lists were subsequently sent to ProteinScapeTM 1.3 (Bruker Daltonics, Bremen, Germany), triggering database searches using ProFound (Version 2002.03.01, Proteometrics LLC) and MASCOT (Version 2.3.02, Matrix Science, London, UK). The following search parameters were selected: fixed cysteine modification with propionamide, variable modification due to methionine oxidation, one maximal missed cleavage sites in case of incomplete trypsin hydrolysis and no details about 2-DE derived protein mass and pI. Using the Score booster function of ProteinScapeTM the mass lists were recalibrated and background masses removed using a list containing 44 masses occurring in a minimum of 10% of generated peak lists. The database searches were run with a mass tolerance of 40 ppm using UniProt’s human complete proteome set (downloaded on 26.10.2012) containing 68.109 protein entries. The used database is a composite database consisting of the UniProtKB entries and a duplicate of the same database, in which the amino acid sequence of each protein entry was randomly shuffled. Proteins reaching Profound score > 1.5 or Mascot score > 64 were considered as identified. Using these criteria one decoy database entry was found by the search engines indicating high confidence of protein identifications. If several database entries of homologues proteins matched these criteria only the entry with the highest score was reported.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2013-05-08</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD000132-4&outputMode=XML&test=no' target="_blank">Submission_2013-07-26_05:37:48.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="http://dx.doi.org/10.6019/PXD000132" target="_blank">http://dx.doi.org/10.6019/PXD000132</a></td></tr><tr><td><b>ReviewLevel</b></td><td>Non peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Supported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Gereon Poschmann</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Homo sapiens (Human);  NCBI TaxID: 9606; </td></tr><tr><td><b>ModificationList</b></td><td>No applicable mass modifications</td></tr><tr><td><b>Instrument</b></td><td>ultraflex TOF/TOF</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2013-01-24 04:06:45</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000132-1&test=no">1</a></td><td >2013-07-26 03:59:33</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000132-1&test=no">1</a></td><td >2014-07-28 00:29:12</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000132-2&test=no">2</a></td><td >2013-07-26 04:00:47</td><td >announced</td><td >missing</td></tr>
<tr><td ><a href="GetDataset?ID=PXD000132-3&test=no">3</a></td><td >2013-07-26 05:36:57</td><td >announced</td><td >missing</td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD000132-4&test=no"><span class='current'>&#9205;</span> 4</a></td><td class="dataset-currentrev">2013-07-26 05:37:48</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">missing</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Poschmann G, Lendzian A, Uszkoreit J, Eisenacher M, Borght AV, Ramaekers FC, Meyer HE, Stühler K, A combination of two electrophoretical approaches for detailed proteome-based characterization of SCLC subtypes. Arch Physiol Biochem, 119(3):114-25(2013) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=23651173" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: SCLC, MALDI, classic and variant, cancer</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Gereon Poschmann</th></tr>
<tr><td>contact affiliation</td><td>Molecular Proteomics Laboratory</td></tr><tr><td>contact email</td><td>gereon.poschmann@hhu.de</td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/2013/05/PXD000132 target="_blank">Dataset FTP location</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/simpleSearch.do?simpleSearchValue=28392" target="_blank">28392</a><ol><li>InstrumentRef: Instrument_1</li><li>Label: [lcvac:Lung SCLC:PMFIdents IEF SDS P]</li><li>ModificationList: No applicable mass modifications, </li><li>Name: lcvac: Lung SCLC:  Peptide Mass Fingerprint Identifications of IEF SDS PAGE</li></ol></li><li><a href="http://www.ebi.ac.uk/pride/simpleSearch.do?simpleSearchValue=28391" target="_blank">28391</a><ol><li>InstrumentRef: Instrument_1</li><li>Label: [lcvac:Lung SCLC:PMFIdents CTAB PAGE]</li><li>ModificationList: No applicable mass modifications, </li><li>Name: lcvac: Lung SCLC:  Peptide Mass Fingerprint Identifications of CTAB PAGE</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>