<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD002779</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD002779</h1>
    
	<div class="dataset-content">
	PXD002779 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Rescuing discarded spectra: Full comprehensive analysis of a minimal proteome.</td></tr><tr><td><b>Description</b></td><td class='breakwords'>A common problem encountered when performing large-scale mass spectrometry proteome analysis is the loss of information due to the high percentage of unassigned spectra. To determine the causes behind this loss we have analyzed the proteome of one of the smallest living bacteria that can be grown axenically, Mycoplasma pneumoniae (729 ORFs). M. pneumoniae cultures were grown in defined media, and their proteomes were analyzed by mass spectrometry. An initial database search with a curated M. pneumoniae protein database left around 78% of the acquired spectra without an assignment. After re-analysis of the data with the PEAKS software and a larger protein database, the percentage of non-assigned spectra was reduced to 27%, thereby increasing the proteome coverage of M. pneumoniae from the initial 60% to over 76%. Nonetheless, 33413 of spectra with assigned amino acid sequences, could not be mapped in any NCBInr database protein sequence. Approximately, 1% of these unassigned peptides corresponded to post-translational modifications and 4% to protein variants. We found that deamidation of Asn affected preferentially to integral membrane proteins and that about 1% of the peptides had repetitions of the same aromatic/hydrophobic amino acid at the N-terminus, or Arg/Lys at the C-terminus. Thus, in an ideal system, we maximized the level of assignment to 73% of the spectra (51453 out of 70040 initial acquired spectra).</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2016-01-05</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD002779-1&outputMode=XML&test=no' target="_blank">Submission_2016-01-05_14:27:42.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Francesco Mattia Mancuso</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Mycoplasma pneumoniae (strain ATCC 29342 / M129);  NCBI TaxID: 272634; </td></tr><tr><td><b>ModificationList</b></td><td>monohydroxylated residue; iodoacetamide derivatized residue</td></tr><tr><td><b>Instrument</b></td><td>LTQ Orbitrap Velos</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2015-08-24 01:31:53</td><td >ID requested</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD002779-1&test=no"><span class='current'>&#9205;</span> 1</a></td><td class="dataset-currentrev">2016-01-05 14:27:43</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev"></td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Lluch-Senar M, Mancuso FM, Climente-González H, Peña-Paz MI, Sabido E, Serrano L, Rescuing discarded spectra: Full comprehensive analysis of a minimal proteome. Proteomics, 16(4):554-63(2016) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=26702875" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>curator keyword: Technical</td></tr><tr><td>submitter keyword: : Mass Spectroscopy, Proteome coverage, translation errors, Post-translational modifications (PTMs), Protein identification, Unassigned spectra</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Eduard Sabidò</th></tr>
<tr><td>contact affiliation</td><td>Proteomics Unit, Centre for Genomic Regulation (CRG), Dr. Aiguader 88, 08003 Barcelona, Spain</td></tr><tr><td>contact email</td><td>eduard.sabido@crg.eu</td></tr><tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Francesco Mattia Mancuso</th></tr>
<tr><td>contact affiliation</td><td>CRG/UPF Proteomics Unit</td></tr><tr><td>contact email</td><td>francescomattia.mancuso@gmail.com</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2016/01/PXD002779 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD002779 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD002779" target="_blank">PXD002779</a><ol><li>Label: PRIDE project</li><li>Name: Rescuing discarded spectra: Full comprehensive analysis of a minimal proteome.</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>