<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD023736</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD023736</h1>
    
	<div class="dataset-content">
	PXD023736 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>A systematic exploration of antimicrobial resistance to beta-lactams, aminoglycosides and fluoroquinolones in Escherichia coli and Klebsiella pneumoniae using proteogenomics</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Background: Antimicrobial resistance is generally studied using a combination of growth inhibition measurements, sometimes in combination with DNA detection methods. However, the actual proteins that cause resistance such as enzymes, efflux pumps and a lack of porins cannot be detected by these methods. Improvements in liquid chromatography (LC) and mass spectrometry (MS) enabled easier and more comprehensive proteome analysis. In the current study, these three methods are combined into a multi-omics approach to analyze resistance against frequently used antibiotics within the beta-lactam, aminoglycoside and fluoroquinolone group in E. coli and K. pneumoniae. Objectives: We aimed to analyze which currently known antimicrobial resistance genes are detected at the protein level using liquid chromatography-mass spectrometry (LC-MS/MS) and to assess whether these could explain beta-lactam, aminoglycoside, and fluoroquinolone resistance in the studied isolates. Furthermore, we aimed to identify significant protein to resistance correlations which have not yet been described and to correlate the abundance of different porins to resistance. Methods: Whole genome sequencing, high-resolution LC-MS/MS and antimicrobial susceptibility testing by broth microdilution were performed for 187 clinical E. coli and K. pneumoniae isolates. Resistance genes and proteins were identified using the Comprehensive Antibiotic Resistance Database (CARD). All proteins were annotated using the NCBI RefSeq database and Prokka. Results & Conclusion: Proteins of small spectrum beta-lactamases, extended spectrum beta-lactamases, AmpC beta-lactamases, carbapenemases, and proteins of 16S ribosomal RNA methyltransferases and aminoglycoside acetyltransferases can be detected in E. coli and K. pneumoniae by LC-MS/MS. The detected mechanisms could explain phenotypic resistance in most of the studied isolates. Differences in the abundance and the primary structure of other proteins such as porins also correlated with resistance. LC-MS/MS is a different and complementary method which can be used to characterize antimicrobial resistance in detail as not only the primary resistance causing mechanisms are detected, but also secondary enhancing resistance mechanisms.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2021-07-09</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD023736.0-2&outputMode=XML&test=no' target="_blank">Submission_2021-07-09_04:27:50.822.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Lennard Dekker</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Escherichia coli;  NCBI TaxID: 562; </td></tr><tr><td><b>ModificationList</b></td><td>iodoacetamide derivatized residue</td></tr><tr><td><b>Instrument</b></td><td>Orbitrap Fusion Lumos</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2021-01-21 05:27:40</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD023736-1&test=no">1</a></td><td >2021-07-08 23:30:12</td><td >announced</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD023736-2&test=no"><span class='current'>&#9205;</span> 2</a></td><td class="dataset-currentrev">2021-07-09 04:27:51</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">2021-07-09: Updated project metadata.</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td><a href="http://dx.doi.org/10.1038/S41598-021-91905-W" target="_blank">10.1038/S41598-021-91905-W</a>;</td></tr><tr><td>Foudraine DE, Strepis N, Stingl C, Ten Kate MT, Verbon A, Klaassen CHW, Goessens WHF, Luider TM, Dekker LJM, Exploring antimicrobial resistance to beta-lactams, aminoglycosides and fluoroquinolones in E. coli and K. pneumoniae using proteogenomics. Sci Rep, 11(1):12472(2021) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=34127720" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: liquid chromatography, mass spectrometry, aminoglycosides, K. pneumoniae, data dependent acquisition, porins, comprehensive antibiotic resistance database, beta-lactams, fluoroquinolones, minimum inhibitory concentrations,E. coli, whole genome sequencing, antimicrobial resistance</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Theo Luider</th></tr>
<tr><td>contact affiliation</td><td>Lab. Neuro-oncologie en klinische en kanker proteomics Department of Neurology Erasmus MC</td></tr><tr><td>contact email</td><td>t.luider@erasmusmc.nl</td></tr><tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Lennard Dekker</th></tr>
<tr><td>contact affiliation</td><td>Erasmus MC</td></tr><tr><td>contact email</td><td>l.dekker@erasmusmc.nl</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2021/07/PXD023736 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD023736 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD023736" target="_blank">PXD023736</a><ol><li>Label: PRIDE project</li><li>Name: A systematic exploration of antimicrobial resistance to beta-lactams, aminoglycosides and fluoroquinolones in Escherichia coli and Klebsiella pneumoniae using proteogenomics</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>