<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD000152</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD000152</h1>
    
	<div class="dataset-content">
	PXD000152 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Conservation of the extended substrate specificity profiles among homologous granzymes across species</td></tr><tr><td><b>Description</b></td><td class='breakwords'>Granzymes are a family of structurally related serine proteases involved in immunity. To date four out of five human granzymes have been assigned orthologues in mice; however for granzyme H, no clear murine orthologue has been suggested and its role in cytotoxicity remains controversial. In this study, we demonstrate that granzyme H is an inefficient cytotoxin. Besides analyzing the substrate specificity profile of granzyme H by means of substrate phage display, we assessed human granzyme H and mouse granzyme C cleavage susceptibility on a proteome-wide level. The extended specificity profiles of granzyme C and granzyme H (i.e. beyond P4-P4') match those previously observed for granzyme B. We demonstrate conservation of these extended specificity profiles among various granzymes since granzyme B cleavage susceptibility of an otherwise granzyme H/C specific cleavage site can be conferred simply by altering the P1-residue to aspartate, the preferred P1-residue of granzyme B. Our results thus indicate a conserved, but hitherto generally underappreciated specificity-determining role of extended protease-substrate contacts in steering cleavage susceptibility.  Bio-IT: The spectra were searched with Mascot Daemon 2.2. The following search parameters were used. Peptide mass tolerance was set at 0.2 Da and peptide fragment mass tolerance at 0.1 Da; with the ESI-QUAD-TOF as selected instrument for peptide fragmentation rules for the Q-TOF Premier data. Endoproteinase semiArg-C/P (i.e., no restriction towards arginine-proline cleavage) was set as enzyme allowing one missed cleavage. Variable modifications were pyroglutamate formation of N-terminal glutamine, pyrocarbamidomethyl formation of N-terminal alkylated cysteine, deamidation of asparagine, acetylation or tri-deuteroacetylation of the alpha-N-terminus. Fixed modifications were methionine oxidation (sulfoxide), carbamidomethyl for cysteine, tri-deuteroacetylation of lysine and for identifying heavy labelled peptides, [13C6] or [13C615N4]-arginine were additionally set as fixed modifications.  The spectra and identifications were stored in ms_lims</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2014-08-08</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD000152-3&outputMode=XML&test=no' target="_blank">Submission_2014-08-08_00:30:44.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="http://dx.doi.org/10.6019/PXD000152" target="_blank">http://dx.doi.org/10.6019/PXD000152</a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Supported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Pieter-Jan De Bock</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Mus musculus (Mouse);  NCBI TaxID: 10090; </td></tr><tr><td><b>ModificationList</b></td><td>L-methionine sulfoxide: 15.994915; 6x(13)C: 4x(15)N labeled L-arginine; 2-pyrrolidone-5-carboxylic acid (Gln): -17.026549; 6x(13)C labeled L-arginine: 6.020129; iodoacetamide derivatized residue: 57.021464; N-acetylated residue: 42.010565; deamidated residue: 0.984016; acetate labeling reagent (N-term) (heavy form: +3amu)</td></tr><tr><td><b>Instrument</b></td><td>instrument model: QToF Global; Waters instrument model</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2013-02-19 08:03:24</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000152-1&test=no">1</a></td><td >2013-11-27 02:43:48</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000152-1&test=no">1</a></td><td >2013-11-27 02:52:20</td><td >announced</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD000152-2&test=no">2</a></td><td >2014-07-28 00:29:57</td><td >announced</td><td >Updated project metadata.</td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD000152-3&test=no"><span class='current'>&#9205;</span> 3</a></td><td class="dataset-currentrev">2014-08-08 00:30:45</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Updated project metadata.</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Plasman K, Maurer-Stroh S, Ahmad J, Hao H, Kaiserman D, Sirota FL, Jonckheere V, Bird PI, Gevaert K, Van Damme P, Conservation of the extended substrate specificity profiles among homologous granzymes across species. Mol Cell Proteomics, 12(10):2921-34(2013) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=23788529" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>curator keyword: Biomedical</td></tr><tr><td>submitter keyword: Granzyme H, granzyme C, human, mouse, LC-MSMS, N-terminal Cofradic</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Pieter-Jan De Bock</th></tr>
<tr><td>contact affiliation</td><td>Biochemistry</td></tr><tr><td>contact email</td><td>pieter-jan.debock@cea.fr</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2013/11/PXD000152 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD000152 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD000152" target="_blank">PXD000152</a><ol><li>Label: PRIDE project</li><li>Name: Conservation of the extended substrate specificity profiles among homologous granzymes across species</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>