<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD015909</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD015909</h1>
    
	<div class="dataset-content">
	PXD015909 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>Integrated bottom-up and top-down proteomics of patient-derived breast tumor xenografts</td></tr><tr><td><b>Description</b></td><td class='breakwords'><p><a href="http://www.mcponline.org/content/early/2015/10/26/mcp.M114.047480.full.pdf+html" target="_blank">Ntai, I., LeDuc, R.D., Fellers, R.T., et al., (2015) Molecular and Cellular Proteomics, Manuscript M114.047480</a></p><p>Bottom-up proteomics relies on the use of proteases and is the method of choice for identifying thousands of protein groups in complex samples. Top-down proteomics has been shown to be robust for direct analysis of small proteins and offers a solution to the peptide-to- protein inference problem inherent with bottom-up approaches. Here, we describe the first large-scale integration of genomic, bottom-up and top-down proteomic data for the comparative analysis of patient-derived mouse xenograft models of basal and luminal B human breast cancer, WHIM2 and WHIM16, respectively. Using these well-characterized xenograft models established by the National Cancer Institute's Clinical Proteomic Tumor Analysis Consortium, we compared and contrasted the performance of bottom-up and top-down proteomics to detect cancer-specific aberrations at the peptide and proteoform levels, and to measure differential expression of proteins and proteoforms. Bottom-up proteomic analysis of the tumor xenografts detected almost 10 times as many coding nucleotide polymorphisms and peptides resulting from novel splice junctions than top-down. For proteins in the range of 0-30 kDa, where quantitation was performed using both approaches, bottom-up proteomics quantified 3,519 protein groups from 49,185 peptides, while top-down proteomics quantified 982 proteoforms mapping to 358 proteins. Examples of both concordant and discordant quantitation were found in an approximately 60:40 ratio, providing a unique opportunity for top-down to fill in missing information. The two techniques showed complementary performance, with bottom-up yielding 8 times more identifications of 0-30 kDa proteins in xenograft proteomes, but failing to detect differences in certain post-translational modifications (PTMs), such as phosphorylation pattern changes of alpha-endosulfine. This work illustrates the potency of a combined bottom-up and top-down proteomics approach to deepen our knowledge of cancer biology, especially when genomic data are available.</p><ul><li>Dataset imported into MassIVE from <a href="https://cptac-data-portal.georgetown.edu/cptac/s/S028">https://cptac-data-portal.georgetown.edu/cptac/s/S028</a> on 08/28/19</li></ul></td></tr><tr><td><b>HostingRepository</b></td><td>MassIVE</td></tr><tr><td><b>AnnounceDate</b></td><td>2020-01-23</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD015909.0-1&outputMode=XML&test=no' target="_blank">Submission_2020-01-23_16:38:36.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Non peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Christopher Kinsinger</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Homo sapiens;  common name: human;  NCBI TaxID: 9606;  scientific name: Mus musculus;  common name: house mouse;  NCBI TaxID: 10090; </td></tr><tr><td><b>ModificationList</b></td><td>Carbamidomethyl; Oxidation</td></tr><tr><td><b>Instrument</b></td><td>TripleTOF 5600; LTQ Orbitrap Elite</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2019-10-18 09:00:52</td><td >ID requested</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD015909-1&test=no"><span class='current'>&#9205;</span> 1</a></td><td class="dataset-currentrev">2020-01-23 16:38:37</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev"></td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>no publication</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>submitter keyword: CPTAC</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Neil L Kelleher</th></tr>
<tr><td>contact affiliation</td><td>Northwestern University</td></tr><tr><td>contact email</td><td>neil.kelleher@norwestern.edu</td></tr><tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Christopher Kinsinger</th></tr>
<tr><td>contact affiliation</td><td>NIH/NCI</td></tr><tr><td>contact email</td><td>Christopher.kinsinger@nih.gov</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=http://massive.ucsd.edu/ProteoSAFe/dataset.jsp?task=5a5ef634200b48658921084641be6770 target="_blank">MassIVE dataset URI</a></td></tr><tr><td><a href=ftp://massive.ucsd.edu/MSV000084247/ target="_blank">Dataset FTP location</a></td></tr></table>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>