<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><link rel="stylesheet" href="/css/isb.css" /><title>ProteomeXchange Dataset PXD004824</title>
</head><body bgcolor="#f4f6f8" marginwidth="0" marginheight="0">	<table width="100%" height="127" background="/images/topbackground.jpg" style="box-shadow: 0 6px 8px 0 rgba(0,0,0,0.3);">	<tr align="center"><td><a href="http://www.proteomexchange.org/" border="0"><img src="/images/proteomexchange.png"></a></td><td><a href="http://proteomecentral.proteomexchange.org/" border="0"><img src="/images/proteomecentral.png"></a></td></tr>	</table>		<!-- Main content area here --><!-- BEGIN main content -->

	 <link rel='stylesheet' id='style-css'  href='../javascript/css/patchwork.css' type='text/css' media='all' />
         <script type="text/javascript" src="/javascript/js/toggle.js"></script>
      
       <div id="main">
       <a class="linkback" href="http://proteomecentral.proteomexchange.org/cgi/GetDataset"> &lt;&lt;&lt; Full experiment listing </a>
       <div id="dataset-primary" class="site-content">
       <h1 class="dataset-title">PXD004824</h1>
    
	<div class="dataset-content">
	PXD004824 is an <b>original dataset</b> announced via ProteomeXchange.<BR><BR><div class='dataset-secthead'>Dataset Summary</div>
<table class='dataset-summary'><tr><td><b>Title</b></td><td>DISMS2: A flexible algorithm for direct proteome-wide distance calculation of LC-MS/MS runs</td></tr><tr><td><b>Description</b></td><td class='breakwords'>The classification of samples on a molecular level has manifold applications, from cancer biology, where the goal is to classify patients according to effective treatments, to phylogenetics to identify evolutionary relationships between species. In such scenarios modern molecular methods are based on the alignment of DNA or amino acid sequences, often only on selected parts of the genome, but also genome-wide comparisons of sequences are performed. Recently proteomics-based approaches have become popular. An established method for the identification of peptides and proteins is liquid chromatography - tandem mass spectrometry (LC-MS/MS). This technique is used to identify protein sequences from tandem mass spectra by means of database searches, given samples with known genome-wide sequence information, and then to apply sequence based methods. Alternatively, de novo peptide sequencing algorithms annotate MS/MS spectra and deduce peptide/protein information without the need of database. A newer approach independent of additional information is to directly compare unidentified tandem mass spectra. The challenge then is to compute the distance between pairwise MS/MS runs consisting of thousands of spectra.</td></tr><tr><td><b>HostingRepository</b></td><td>PRIDE</td></tr><tr><td><b>AnnounceDate</b></td><td>2017-03-13</td></tr>
                <tr><td><b>AnnouncementXML</b></td><td><a href='GetDataset?ID=PXD004824-2&outputMode=XML&test=no' target="_blank">Submission_2017-10-02_07:59:26.xml</a></td></tr>
                <tr><td><b>DigitalObjectIdentifier</b></td><td><a href="" target="_blank"></a></td></tr><tr><td><b>ReviewLevel</b></td><td>Peer-reviewed dataset</td></tr><tr><td><b>DatasetOrigin</b></td><td>Original dataset</td></tr><tr><td><b>RepositorySupport</b></td><td>Unsupported dataset by repository</td></tr><tr><td><b>PrimarySubmitter</b></td><td>Laxmikanth Kollipara</td></tr><tr><td><b>SpeciesList</b></td><td> scientific name: Mus musculus (Mouse);  NCBI TaxID: 10090;  scientific name: Caenorhabditis elegans;  NCBI TaxID: 6239;  scientific name: Homo sapiens (Human);  NCBI TaxID: 9606;  scientific name: Saccharomyces cerevisiae (Baker's yeast);  NCBI TaxID: 4932;  scientific name: Drosophila melanogaster (Fruit fly);  NCBI TaxID: 7227; </td></tr><tr><td><b>ModificationList</b></td><td>monohydroxylated residue; iodoacetamide derivatized residue</td></tr><tr><td><b>Instrument</b></td><td>Q Exactive</td></tr></table>
<div class='dataset-secthead'>Dataset History</div>
<table class="dataset-summary">
<tr><th>Revision</th><th>Datetime</th><th>Status</th><th>ChangeLog Entry</th></tr>
<tr><td >0</td><td >2016-08-16 23:56:12</td><td >ID requested</td><td ></td></tr>
<tr><td ><a href="GetDataset?ID=PXD004824-1&test=no">1</a></td><td >2017-03-13 04:00:27</td><td >announced</td><td ></td></tr>
<tr><td class="dataset-currentrev"><a href="GetDataset?ID=PXD004824-2&test=no"><span class='current'>&#9205;</span> 2</a></td><td class="dataset-currentrev">2017-10-02 07:59:27</td><td class="dataset-currentrev">announced</td><td class="dataset-currentrev">Updated publication reference for PubMed record(s): 28253837, 28959885.</td></tr>
</table>
<div class='dataset-secthead'>Publication List </div>
<table class='dataset-summary'>
<tr><td>Rieder V, Blank-Landeshammer B, Stuhr M, Schell T, Biß K, Kollipara L, Meyer A, Pfenninger M, Westphal H, Sickmann A, Rahnenführer J, DISMS2: A flexible algorithm for direct proteome- wide distance calculation of LC-MS/MS runs. BMC Bioinformatics, 18(1):148(2017) [<a href="https://www.ncbi.nlm.nih.gov/pubmed?term=28253837" target="_blank">pubmed</a>]</td></tr></table>
<div class='dataset-secthead'>Keyword List </div>
<table class='dataset-summary'>
<tr><td>curator keyword: Technical</td></tr><tr><td>submitter keyword: Distance measurements, Spectral comparisions, LC-MS/MS</td></tr></table>
<div class='dataset-secthead'>Contact List </div>
<table class='dataset-summary'>
<tr><th colspan='2'>Albert Sickmann</th></tr>
<tr><td>contact affiliation</td><td>Leibniz-Institut für Analytische Wissenschaften – ISAS – e.V. Otto-Hahn-Straße 6b 44227 Dortmund, Germany</td></tr><tr><td>contact email</td><td>sickmann@isas.de</td></tr><tr><td>lab head</td><td></td></tr><tr><th colspan='2'>Laxmikanth Kollipara</th></tr>
<tr><td>contact affiliation</td><td>Leibniz – Institut für Analytische Wissenschaften - ISAS - e.V.</td></tr><tr><td>contact email</td><td>laxmikanth.kollipara@isas.de</td></tr><tr><td>dataset submitter</td><td></td></tr></table>
<div class='dataset-secthead'>Full Dataset Link List </div>
<table class='dataset-summary'>
<tr><td><a href=ftp://ftp.pride.ebi.ac.uk/pride/data/archive/2017/03/PXD004824 target="_blank">Dataset FTP location</a></td></tr><tr><td><a href=http://www.ebi.ac.uk/pride/archive/projects/PXD004824 target="_blank">PRIDE project URI</a></td></tr></table>

		 <div class='dataset-secthead'>Repository Record List </div>
                 <div id="less" onclick='toggle("less","more","repositoryRecordList")' title='Expand Record List'>[ + ]</div>
		 <div id="more" onclick='toggle("more","less","repositoryRecordList")' title='Hide Record List' style='display: none'>[ - ]</div>
		 <ul id="repositoryRecordList" style='display: none'>
        <li>PRIDE<ol><li><a href="http://www.ebi.ac.uk/pride/archive/projects/PXD004824" target="_blank">PXD004824</a><ol><li>Label: PRIDE project</li><li>Name: DISMS2: A flexible algorithm for direct proteome-wide distance calculation of LC-MS/MS runs</li></ol></li></li></ul>

      
	 </div><!-- #content -->
	 </div><!-- #primary .site-content -->
	 </div><!-- #main -->
      <!-- END main content -->	<!-- Begin Footer -->	<br clear="all">	<div class="footer">	<table width="100%"><tr>	<td style="vertical-align:top; text-align:left; padding: 10px;">If you have a question or comment about ProteomeXchange, please <a href="http://www.proteomexchange.org/contact">contact us</a>!<BR>	<a href="http://www.proteomexchange.org/subscribe/index.html"><img width="70" height="20" src="/images/subscribe_button-small.jpg"><font size="+1" color="#aa0000"> to receive all new ProteomeXchange dataset release announcements!</font></a>	</td>	<td style="text-align:right; padding: 10px;">	<a class="logolink" target="_blank" href="http://www.ebi.ac.uk/"><img src="/images/ebi_footer_logo.png"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://www.isbscience.org/"><img src="/images/isb.jpg"></a>&nbsp;&nbsp;&nbsp;&nbsp;	<a class="logolink" target="_blank" href="http://ec.europa.eu/research/fp7/index_en.cfm"><img src="/images/capacities.png"></a>	</td></tr></table>	</div>				</body></html>